#MuSiC2: cell type deconvolution for multi-condition bulk RNA-seq data
library(SingleCellExperiment)
#BiocManager::install(c("scater","scran","limma","data.table","svd","Rtsne","SC3"))
library(scater)
library(scran)
library(limma)
library(data.table)
library(svd)
library(Rtsne)
library(SC3)
library(edgeR)
library(openxlsx)
library(ggrepel)
library(ggplot2)
library(DBI)
library(org.Hs.eg.db)
library(dplyr)
library("pcaExplorer")
#BiocManager::install("GEOquery")
library(GEOquery)
#install TOAST
#if (!requireNamespace("BiocManager", quietly = TRUE))
#  install.packages("BiocManager")
#BiocManager::install(version='3.16')
#BiocManager::install("TOAST")
library(TOAST)
# install devtools if necessary
#if (!"devtools" %in% rownames(installed.packages())) {
#  install.packages('devtools')
#}
#BiocManager::install("SingleCellExperiment")
# install the MuSiC package
#devtools::install_github('xuranw/MuSiC')
# load
library(MuSiC)
#install.packages("remotes")
#remotes::install_github("renozao/xbioc")

# install the MuSiC2 package
if (!"MuSiC2" %in% rownames(installed.packages())) {
  devtools::install_github('Jiaxin-Fan/MuSiC2')
}
# load
library(MuSiC2)
#BiocManager::install("Biobase")
library("Biobase")
# Enter commands in R (or R studio, if installed)
#install.packages('Seurat')
library(Seurat)
#install.packages("tidyverse")
library("tidyverse")

#Human brain single-cell gene expression data from the middle temporal gyrus generated by Darmanis et al.
#were downloaded as count-level data
#from https://github.com/VCCRI/CIDR-examples/tree/master/Brain
### store sample.info and expression matrix
## Preprocess data from Darmanis et al.
# read in
darmanis <- read.csv("data/Supp_Figure2/brainTags.csv")
rownames(darmanis) <- darmanis[,1]
darmanis <- darmanis[,-1]

# annotate
darmanis_meta <- read.table("data/Supp_Figure2/SraRunTable.txt", sep = "\t", header = 1)
darmanis_meta <- darmanis_meta[-grep("hybrid", darmanis_meta$cell_type_s),] # "hybrid" refers to transcriptomes that likely contain >1 cell
rownames(darmanis_meta) <- darmanis_meta$Sample_Name_s
darmanis <- darmanis[,rownames(darmanis_meta)]
darmanis_meta$cell_type_s <- as.factor(darmanis_meta$cell_type_s)
levels(darmanis_meta$cell_type_s) <- c("Astrocytes", "Endothelia", "FoetalQuiescent", "FoetalRep", "Microglia", "Neurons", "Oligodendrocytes", "OPC")
darmanis_meta$cell_type_s <- as.character(darmanis_meta$cell_type_s)

# filter to five key cell-types
keep <- which(darmanis_meta$cell_type_s %in% c("Astrocytes", "Endothelia", "Microglia", "Neurons", "Oligodendrocytes","OPC"))
darmanis <- darmanis[,keep]
darmanis_meta <- darmanis_meta[keep,]
darmanis_sce <- SingleCellExperiment(
  assays = list(counts = as.matrix(darmanis)),
  colData = darmanis_meta
)
darmanis_sce

#Hippocampus
#Sample Analysis
## Bulk RNA-seq data
#Building an ExpressionSet From Scratch
exprsFile_HH<-"data/Supp_Figure2/cpm_y_hth_filt_coding22_after_PCA.txt"
exprs_HH <- as.matrix(read.table(exprsFile_HH, header=TRUE, sep="\t",row.names=1,as.is=TRUE))
class(exprs_HH)
dim(exprs_HH)
colnames(exprs_HH)
head(exprs_HH[,1:5])
minimalSet_HH <- ExpressionSet(assayData=exprs_HH)
pDataFile_HH <-"data/Supp_Figure2/pData_HH.txt"
pData_HH <- read.table(pDataFile_HH,row.names=1, header=TRUE, sep="\t")
dim(pData_HH)
rownames(pData_HH)
summary(pData_HH)
all(rownames(pData_HH)==colnames(exprs_HH))
metadata_HH <- data.frame(labelDescription=c("Patient group","Gender","Age","Race","RIN values"),row.names=c("Group", "Sex", "Age","Race","RIN"))
phenoData_HH <- new("AnnotatedDataFrame",data=pData_HH, varMetadata=metadata_HH)
phenoData_HH

annotation <- "TxDb.Hsapiens.UCSC.hg38.knownGene"

ExpressionSet_HH <- ExpressionSet(assayData=exprs_HH,phenoData=phenoData_HH,annotation = "TxDb.Hsapiens.UCSC.hg38.knownGene")
ExpressionSet_HH

table(ExpressionSet_HH$Group)
bulk.control.mtx_HH = exprs(ExpressionSet_HH)[,  ExpressionSet_HH$Group == 'Ctrl']
bulk.case.mtx_HH = exprs(ExpressionSet_HH)[, ExpressionSet_HH$Group == 'Down']

set.seed(1234)
est_DM_HH = music2_prop_t_statistics(bulk.control.mtx = bulk.control.mtx_HH, bulk.case.mtx = bulk.case.mtx_HH,
                                     sc.sce = darmanis_sce, clusters = 'cell_type_s', samples = 'Sample_Name_s',
                                     select.ct = c('Astrocytes','Neurons','Oligodendrocytes','Microglia','Endothelia','OPC'),
                                     n_resample=20, sample_prop=0.5,cutoff_c=0.05,cutoff_r=0.01)
est.prop_DM_HH = est_DM_HH$Est.prop
est.DE.genes_DM_HH=est_DM_HH$DE.genes
write.table(est.DE.genes_DM_HH,"output/Darmanis_human_hippo_bulk_RNA_DE_genes.txt",sep = "\t")
write.table(est.prop_DM_HH,"output/Darmanis_human_hippo_bulk_RNA_proportion.txt",sep = "\t")

#cortex
exprsFile_HC <-"data/Supp_Figure2/cpm_y_cth_filt_coding22_after_PCA.txt"
exprs_HC <- as.matrix(read.table(exprsFile_HC, header=TRUE, sep="\t",row.names=1,as.is=TRUE))
class(exprs_HC)
dim(exprs_HC)
colnames(exprs_HC)
head(exprs_HC[,1:5])
minimalSet_HC <- ExpressionSet(assayData=exprs_HC)
pDataFile_HC <-"data/Supp_Figure2/pData_HC.txt"
pData_HC <- read.table(pDataFile_HC,row.names=1, header=TRUE, sep="\t")
dim(pData_HC)
rownames(pData_HC)
summary(pData_HC)
all(rownames(pData_HC)==colnames(exprs_HC))
metadata_HC <- data.frame(labelDescription=c("Patient group","Gender","Age","Race","RIN values"),row.names=c("Group", "Sex", "Age","Race","RIN"))
phenoData_HC <- new("AnnotatedDataFrame",data=pData_HC, varMetadata=metadata_HC)
phenoData_HC
annotation <- "TxDb.Hsapiens.UCSC.hg38.knownGene"
ExpressionSet_HC<- ExpressionSet(assayData=exprs_HC,phenoData=phenoData_HC,annotation = "TxDb.Hsapiens.UCSC.hg38.knownGene")
ExpressionSet_HC
table(ExpressionSet_HC$Group)
bulk.control.mtx_HC = exprs(ExpressionSet_HC)[,  ExpressionSet_HC$Group == 'Ctrl']
bulk.case.mtx_HC = exprs(ExpressionSet_HC)[, ExpressionSet_HC$Group == 'Down']

set.seed(1234)
est_DM_HC = music2_prop_t_statistics(bulk.control.mtx = bulk.control.mtx_HC, bulk.case.mtx = bulk.case.mtx_HC,
                                     sc.sce = darmanis_sce, clusters = 'cell_type_s', samples = 'Sample_Name_s',
                                     select.ct = c('Astrocytes','Neurons','Oligodendrocytes','Microglia','Endothelia'),
                                     n_resample=20, sample_prop=0.5,cutoff_c=0.05,cutoff_r=0.01)
est.prop_DM_HC = est_DM_HC$Est.prop
est.DE.genes_DM_HC=est_DM_HC$DE.genes
write.table(est.DE.genes_DM_HC,"output/Darmanis_human_cortex_bulk_RNA_DE_genes.txt",sep = "\t")
write.table(est.prop_DM_HC,"output/Darmanis_human_cortex_bulk_RNA_proportion.txt",sep = "\t")

#Supp Figure2A
#Hippocampus
cell_proportion_DM_music2_hippo<-read.xlsx("data/Supp_Figure2/deconvolution_proportions.xlsx",sheet=4)
CP_DM_M2_hippo_01 <- ggplot(cell_proportion_DM_music2_hippo, aes(Celltype,Proportion,fill=Group))+
  geom_bar(stat="identity",color="black", width = 0.7, position = "dodge") +
  geom_errorbar(aes(ymin=Proportion-sem, ymax=Proportion+sem), width=.1,position=position_dodge(.7))+
  scale_fill_manual(values = c("#B9D47A","#C2A3DC")) +
  ylim(0,1.0)+
  theme_bw()+
  theme(legend.position="none",axis.text.x = element_text(angle=90, vjust=0.6,color ="black"),text=element_text(size=10),
        axis.text.y = element_text(color="black"),panel.background = element_blank(),panel.grid.major = element_blank(),
        panel.grid.minor = element_blank()) +
  labs(x="Cell types",y="Proportion")

#cortex
cell_proportion_DM_music2_cortex<-read.xlsx("data/Supp_Figure2/deconvolution_proportions.xlsx",sheet=5)
CP_DM_M2_cortex_01 <- ggplot(cell_proportion_DM_music2_cortex, aes(Celltype,Proportion,fill=Group))+
  geom_bar(stat="identity",color="black", width = 0.7, position = "dodge") +
  geom_errorbar(aes(ymin=Proportion-sem, ymax=Proportion+sem), width=.1,position=position_dodge(.7))+
  scale_fill_manual(values = c("#B9D47A","#C2A3DC")) +
  ylim(0,1.0)+
  theme_bw()+
  theme(legend.position="none",axis.text.x = element_text(angle=90, vjust=0.6,color ="black"),text=element_text(size=10),
        axis.text.y = element_text(color="black"),panel.background = element_blank(),panel.grid.major = element_blank(),
        panel.grid.minor = element_blank()) +
  labs(x="Cell types",y="Proportion")

#Perform differential expression analysis including the significant changing celltype proportions as covariates


#Supp Figure2B
#Hippocampus

human_hippo_DM_M2_overlap_DE<-read.xlsx("data/Supp_Figure2/Scatter_compare_original_analysis.xlsx",sheet=1)
reg<-lm(Original ~ Corrected, data = human_hippo_DM_M2_overlap_DE)
reg

human_hippo_DM_M2_overlap_DE_plot<-ggplot(human_hippo_DM_M2_overlap_DE, aes(x = Corrected, y = Original)) +
  geom_point(alpha=1, size=2) +
  #  scale_color_manual(values = c("#EF233C","#CED4DA","#05668D","#FB8500")) +
  xlab("Corrected logFC") + ylab("Original logFC")+
  theme_bw() +
  # xlim(-5,5)+
  #  ylim(-2,4)+
  theme(legend.position="none",text=element_text(size=18),axis.text.x = element_text(color = "black"),axis.text.y = element_text(color="black"),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())+
  geom_abline(intercept = -0.03175, slope = 0.8280, color="black", linetype="dashed", size=0.5)

cor.test(human_hippo_DM_M2_overlap_DE$Corrected,human_hippo_DM_M2_overlap_DE$Original)

#cortex
human_cortex_DM_M2_overlap_DE<-read.xlsx("data/Supp_Figure2/Scatter_compare_original_analysis.xlsx",sheet=2)
reg<-lm(Original ~ Corrected, data = human_cortex_DM_M2_overlap_DE)
reg

human_cortex_DM_M2_overlap_DE_plot<-ggplot(human_cortex_DM_M2_overlap_DE, aes(x = Corrected, y = Original)) +
  geom_point(alpha=1, size=2) +
  #  scale_color_manual(values = c("#EF233C","#CED4DA","#05668D","#FB8500")) +
  xlab("Corrected logFC") + ylab("Original logFC")+
  theme_bw() +
  # xlim(-5,5)+
  #  ylim(-2,4)+
  theme(legend.position="none",text=element_text(size=18),axis.text.x = element_text(color = "black"),axis.text.y = element_text(color="black"),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())+
  geom_abline(intercept = 0.08981, slope = 0.90582, color="black", linetype="dashed", size=0.5)

cor.test(human_cortex_DM_M2_overlap_DE$Corrected,human_cortex_DM_M2_overlap_DE$Original)
