<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="neurodevdisorder" />

<meta name="date" content="2022-07-15" />

<title>differential_expression_analysis</title>

<script src="site_libs/header-attrs-2.13/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">HumanMultiomics</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">differential_expression_analysis</h1>
<h4 class="author">neurodevdisorder</h4>
<h4 class="date">2022-07-15</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-07-15
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>HumanMultiomics/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version 1.7.0). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220715code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20220715)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220715code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220715)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong9bf2d51"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 9bf2d51 </a>
</p>
</div>
<div id="strongRepositoryversionstrong9bf2d51" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 9bf2d51. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    analysis/.DS_Store

Untracked files:
    Untracked:  HumanMultiomics.R
    Untracked:  data/human_hippo_samplesheet.xlsx
    Untracked:  data/raw_counts_human_hippo_matrix_coding.txt
    Untracked:  figure/
    Untracked:  topHitshuman_hippo.bed

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/differential_expression_analysis.Rmd</code>) and HTML (<code>docs/differential_expression_analysis.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
9bf2d51
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Differential expression analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
80ecc5e
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
d1cf5b7
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Differential expression analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
fc2eaa2
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
ea2d35b
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Differential expression analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
09605c0
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
fb514a1
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
7d8790a
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
e1c294a
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ac97638
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
e1a04c1
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Differential expression analysis
</td>
</tr>
<tr>
<td>
html
</td>
<td>
e6b258c
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
155f31d
</td>
<td>
neurodevdisorder
</td>
<td>
2022-07-15
</td>
<td>
Differential expression analysis
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>This is the differential expression analysis from the total RNA sequencing experiment performed on postmortem hippocampus and cortex obtained from Down syndrome and control individuals.</p>
<div id="read-the-raw-count-matrix-for-hippocampus-samples" class="section level3">
<h3>Read the raw count matrix for hippocampus samples</h3>
<pre><code>                 C1  C2   C3   C4  C5  C6   C7   C8  C9 C10 C11  Ds1 Ds2 Ds3
ENSG00000186092   0   0    0    0   0   0    0    0   0   0   0    0   0   0
ENSG00000284733   0   0    0    0   0   0    0    0   0   0   0    0   0   0
ENSG00000284662   0   0    0    0   0   0    0    0   0   0   0    0   0   0
ENSG00000187634   9   9   36   40   8  11   12   69   6  28  12   26  40  29
ENSG00000188976 496 624 1482 1417 558 454 1085 1249 270 477 799 1313 351 629
ENSG00000187961  13  20   89   66  12  34   82  131  13  30  25  105  44  45
                 Ds4 Ds5 Ds6 Ds7 Ds8  Ds9
ENSG00000186092    0   0   0   0   0    0
ENSG00000284733    0   0   0   0   0    0
ENSG00000284662    0   0   0   0   0    0
ENSG00000187634   46  47  21  41  26   22
ENSG00000188976 1051 798 755 570 887 1260
ENSG00000187961   95  98  82  47  32  100</code></pre>
<pre><code> [1] Cont Cont Cont Cont Cont Cont Cont Cont Cont Cont Cont DS   DS   DS   DS  
[16] DS   DS   DS   DS   DS  
Levels: Cont DS</code></pre>
<div id="visualize-the-log-counts-per-million-cpm-as-a-heatmap" class="section level4">
<h4>Visualize the log counts per million (cpm) as a heatmap</h4>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="perform-pca" class="section level4">
<h4>Perform PCA</h4>
<pre><code>[1] TRUE</code></pre>
<pre><code>-- removing the lower 10% of variables based on variance</code></pre>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-5-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="filter-the-data-based-on-heatmap-pca-and-rin-values" class="section level4">
<h4>Filter the data based on heatmap, PCA and RIN values</h4>
<pre><code>                 C1  C2   C3   C4  C5 C11  Ds1 Ds3  Ds4 Ds5 Ds8  Ds9
ENSG00000186092   0   0    0    0   0   0    0   0    0   0   0    0
ENSG00000284733   0   0    0    0   0   0    0   0    0   0   0    0
ENSG00000284662   0   0    0    0   0   0    0   0    0   0   0    0
ENSG00000187634   9   9   36   40   8  12   26  29   46  47  26   22
ENSG00000188976 496 624 1482 1417 558 799 1313 629 1051 798 887 1260
ENSG00000187961  13  20   89   66  12  25  105  45   95  98  32  100</code></pre>
<pre><code> [1] Cont Cont Cont Cont Cont Cont DS   DS   DS   DS   DS   DS  
Levels: Cont DS</code></pre>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-6-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>####Perform PCA on the filtered samples</p>
<pre><code>[1] TRUE</code></pre>
<pre><code>-- removing the lower 10% of variables based on variance</code></pre>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-7-1.png" width="672" style="display: block; margin: auto;" /><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-7-2.png" width="672" style="display: block; margin: auto;" /></p>
</div>
<div id="filter-the-genes-based-on-their-expression-in-n-1-samples" class="section level4">
<h4>Filter the genes based on their expression in n-1 samples</h4>
<pre><code>keep_hth_filt_coding2
FALSE  TRUE 
 5747 14221 </code></pre>
</div>
<div id="perform-normalization-using-tmm" class="section level4">
<h4>Perform normalization using TMM</h4>
</div>
<div id="estimate-differential-expression-using-glm" class="section level4">
<h4>Estimate differential expression using GLM</h4>
<pre><code>    group lib.size norm.factors
C1   Cont 26747264    0.9014953
C2   Cont 28753065    0.9295325
C3   Cont 31216269    1.0367848
C4   Cont 34181183    0.9965316
C5   Cont 24231673    0.9048995
C11  Cont 23897314    0.9656747
Ds1    DS 27072147    1.0308510
Ds3    DS 17654033    1.1121233
Ds4    DS 27713628    1.0497314
Ds5    DS 25307466    1.1031263
Ds8    DS 26297451    0.9796220
Ds9    DS 24491826    1.0163664</code></pre>
<pre><code>Iteration 1: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.
Iteration 2: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.
Iteration 3: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.
Iteration 4: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.
Iteration 5: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.
Iteration 6: Re-fitting GLM. Re-estimating trended dispersion.
Re-estimating tagwise dispersion.</code></pre>
</div>
<div id="number-of-differentially-expressed-genes-in-the-hippocampus" class="section level4">
<h4>Number of differentially expressed genes in the hippocampus</h4>
<pre><code>       group_hth_filt_coding2DS
Down                       2205
NotSig                     9394
Up                         2622</code></pre>
</div>
<div id="here-we-annotated-the-data-from-the-previous-differential-expression-analysis-using-ensembl-information-for-each-gene-and-mapped-the-degs-onto-their-chromosomal-location." class="section level4">
<h4>Here, we annotated the data from the previous differential expression analysis using ENSEMBL information for each gene and mapped the DEGs onto their chromosomal location.</h4>
<pre><code>[1] &quot;ENSG00000188976&quot; &quot;ENSG00000187961&quot; &quot;ENSG00000188290&quot; &quot;ENSG00000187608&quot;
[5] &quot;ENSG00000188157&quot; &quot;ENSG00000131591&quot;</code></pre>
<pre><code>&#39;select()&#39; returned 1:many mapping between keys and columns</code></pre>
<pre><code>GRanges object with 4809 ranges and 12 metadata columns:
                 seqnames            ranges strand |     logFC    logCPM
                    &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt;
  339451             chr1     960584-965719      + |  1.444466   1.13402
  375790             chr1   1020120-1056118      + |  1.428524   4.58160
   54991             chr1   1081818-1116361      - |  1.179575   2.41598
   51150             chr1   1216909-1232067      - |  0.452477   5.35276
    6339             chr1   1280436-1292029      + |  1.051756   2.24617
     ...              ...               ...    ... .       ...       ...
   54058            chr21 46300181-46323875      - |  1.744116   1.84115
    5116            chr21 46324141-46445769      + |  0.772067   6.81541
   23181            chr21 46458891-46570015      + |  0.921378   6.25343
    3275            chr21 46635595-46665124      + |  0.485232   6.94981
  389831 chrUn_GL000195v1       42939-49164      - | -1.191513   2.15811
                LR      PValue         FDR         ENSEMBL    ENTREZID
         &lt;numeric&gt;   &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt;
  339451  12.01408 5.28001e-04 0.003696271 ENSG00000187961      339451
  375790  16.37063 5.20862e-05 0.000684321 ENSG00000188157      375790
   54991  11.76935 6.02139e-04 0.004066014 ENSG00000131591       54991
   51150   7.51441 6.12073e-03 0.023149708 ENSG00000078808       51150
    6339   8.46603 3.61840e-03 0.015531915 ENSG00000162572        6339
     ...       ...         ...         ...             ...         ...
   54058  24.20717 8.65097e-07 3.45406e-05 ENSG00000160298       54058
    5116  25.68207 4.02551e-07 1.96724e-05 ENSG00000160299        5116
   23181  33.96882 5.60024e-09 7.80794e-07 ENSG00000160305       23181
    3275  11.57174 6.69616e-04 4.42441e-03 ENSG00000160310        3275
  389831   6.09842 1.35303e-02 4.21130e-02 ENSG00000276256      389831
              SYMBOL               GENENAME         CHR     UNIPROT       ALIAS
         &lt;character&gt;            &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt;
  339451      KLHL17 kelch like family me..           1      Q6TDP4          AF
  375790        AGRN                  agrin           1      O00468       AGRIN
   54991    C1orf159 chromosome 1 open re..           1      Q96HA4    C1orf159
   51150        SDF4 stromal cell derived..           1  A0A024R084       Cab45
    6339      SCNN1D sodium channel epith..           1      A6NNF7       ENaCd
     ...         ...                    ...         ...         ...         ...
   54058    C21orf58 chromosome 21 open r..          21      P58505    C21orf58
    5116        PCNT            pericentrin          21      O95613         KEN
   23181       DIP2A disco interacting pr..          21      Q14689   C21orf106
    3275       PRMT2 protein arginine met..          21      P55345     HRMT1L1
  389831   LOC389831 uncharacterized LOC3..          14        &lt;NA&gt;   LOC389831
  -------
  seqinfo: 595 sequences (1 circular) from hg38 genome</code></pre>
<pre><code>GRanges object with 4809 ranges and 12 metadata columns:
        seqnames              ranges strand |     logFC    logCPM        LR
           &lt;Rle&gt;           &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;
  64005     chr7   44962662-44979088      - |   3.67879  0.606683   66.0467
   7052    chr20   38127385-38166578      - |   2.25840  4.274056   65.9522
   3339     chr1   21822244-21937310      - |   2.44714  3.740847   65.5580
     12    chr14   94592058-94624646      + |   3.16526  0.988412   64.9642
   3691    chr17   75721328-75757818      + |   2.18007  5.528861   64.2538
    ...      ...                 ...    ... .       ...       ...       ...
  10745     chr1 113696831-113759489      - | -0.361545   5.48447   5.71151
   8495    chr11     7513298-7657127      + |  0.496606   4.93888   5.71125
  54039    chr21   45643694-45942454      + |  0.639404   3.11438   5.70519
  57592     chr1 151281618-151292176      + |  0.795094   3.55223   5.70267
   8708     chr2 167293171-167558333      + | -0.444137   5.26416   5.70133
             PValue         FDR         ENSEMBL    ENTREZID      SYMBOL
          &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt; &lt;character&gt;
  64005 4.40354e-16 2.67493e-12 ENSG00000136286       64005       MYO1G
   7052 4.61983e-16 2.67493e-12 ENSG00000198959        7052        TGM2
   3339 5.64291e-16 2.67493e-12 ENSG00000142798        3339       HSPG2
     12 7.62729e-16 2.71169e-12 ENSG00000196136          12    SERPINA3
   3691 1.09383e-15 3.11108e-12 ENSG00000132470        3691       ITGB4
    ...         ...         ...             ...         ...         ...
  10745   0.0168541   0.0496926 ENSG00000116793       10745       PHTF1
   8495   0.0168566   0.0496926 ENSG00000166387        8495     PPFIBP2
  54039   0.0169148   0.0498541 ENSG00000183570       54039       PCBP3
  57592   0.0169391   0.0499154 ENSG00000143373       57592      ZNF687
   8708   0.0169521   0.0499431 ENSG00000172318        8708     B3GALT1
                      GENENAME         CHR     UNIPROT       ALIAS
                   &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt;
  64005              myosin IG           7      B0I1T2         HA2
   7052     transglutaminase 2          20      P21980        G(h)
   3339 heparan sulfate prot..           1      P98160        HSPG
     12 serpin family A memb..          14  A0A024R6P0        AACT
   3691 integrin subunit bet..          17  A0A024R8T0       CD104
    ...                    ...         ...         ...         ...
  10745 putative homeodomain..           1      Q9UMS5        PHTF
   8495 PPFIA binding protei..          11      Q8ND30       Cclp1
  54039 poly(rC) binding pro..          21      P57721   ALPHA-CP3
  57592 zinc finger protein ..           1      Q8N1G0        PDB6
   8708 beta-1,3-galactosylt..           2      Q9Y5Z6 beta3Gal-T1
  -------
  seqinfo: 595 sequences (1 circular) from hg38 genome</code></pre>
<pre><code>  [1] &quot;chr1&quot;                    &quot;chr2&quot;                   
  [3] &quot;chr3&quot;                    &quot;chr4&quot;                   
  [5] &quot;chr5&quot;                    &quot;chr6&quot;                   
  [7] &quot;chr7&quot;                    &quot;chr8&quot;                   
  [9] &quot;chr9&quot;                    &quot;chr10&quot;                  
 [11] &quot;chr11&quot;                   &quot;chr12&quot;                  
 [13] &quot;chr13&quot;                   &quot;chr14&quot;                  
 [15] &quot;chr15&quot;                   &quot;chr16&quot;                  
 [17] &quot;chr17&quot;                   &quot;chr18&quot;                  
 [19] &quot;chr19&quot;                   &quot;chr20&quot;                  
 [21] &quot;chr21&quot;                   &quot;chr22&quot;                  
 [23] &quot;chrX&quot;                    &quot;chrY&quot;                   
 [25] &quot;chrM&quot;                    &quot;chr1_GL383518v1_alt&quot;    
 [27] &quot;chr1_GL383519v1_alt&quot;     &quot;chr1_GL383520v2_alt&quot;    
 [29] &quot;chr1_KI270706v1_random&quot;  &quot;chr1_KI270707v1_random&quot; 
 [31] &quot;chr1_KI270708v1_random&quot;  &quot;chr1_KI270709v1_random&quot; 
 [33] &quot;chr1_KI270710v1_random&quot;  &quot;chr1_KI270711v1_random&quot; 
 [35] &quot;chr1_KI270712v1_random&quot;  &quot;chr1_KI270713v1_random&quot; 
 [37] &quot;chr1_KI270714v1_random&quot;  &quot;chr1_KI270759v1_alt&quot;    
 [39] &quot;chr1_KI270760v1_alt&quot;     &quot;chr1_KI270761v1_alt&quot;    
 [41] &quot;chr1_KI270762v1_alt&quot;     &quot;chr1_KI270763v1_alt&quot;    
 [43] &quot;chr1_KI270764v1_alt&quot;     &quot;chr1_KI270765v1_alt&quot;    
 [45] &quot;chr1_KI270766v1_alt&quot;     &quot;chr1_KI270892v1_alt&quot;    
 [47] &quot;chr1_KN196472v1_fix&quot;     &quot;chr1_KN196473v1_fix&quot;    
 [49] &quot;chr1_KN196474v1_fix&quot;     &quot;chr1_KN538360v1_fix&quot;    
 [51] &quot;chr1_KN538361v1_fix&quot;     &quot;chr1_KQ031383v1_fix&quot;    
 [53] &quot;chr1_KQ458382v1_alt&quot;     &quot;chr1_KQ458383v1_alt&quot;    
 [55] &quot;chr1_KQ458384v1_alt&quot;     &quot;chr1_KQ983255v1_alt&quot;    
 [57] &quot;chr1_KV880763v1_alt&quot;     &quot;chr1_KZ208904v1_alt&quot;    
 [59] &quot;chr1_KZ208905v1_alt&quot;     &quot;chr1_KZ208906v1_fix&quot;    
 [61] &quot;chr1_KZ559100v1_fix&quot;     &quot;chr2_GL383521v1_alt&quot;    
 [63] &quot;chr2_GL383522v1_alt&quot;     &quot;chr2_GL582966v2_alt&quot;    
 [65] &quot;chr2_KI270715v1_random&quot;  &quot;chr2_KI270716v1_random&quot; 
 [67] &quot;chr2_KI270767v1_alt&quot;     &quot;chr2_KI270768v1_alt&quot;    
 [69] &quot;chr2_KI270769v1_alt&quot;     &quot;chr2_KI270770v1_alt&quot;    
 [71] &quot;chr2_KI270771v1_alt&quot;     &quot;chr2_KI270772v1_alt&quot;    
 [73] &quot;chr2_KI270773v1_alt&quot;     &quot;chr2_KI270774v1_alt&quot;    
 [75] &quot;chr2_KI270775v1_alt&quot;     &quot;chr2_KI270776v1_alt&quot;    
 [77] &quot;chr2_KI270893v1_alt&quot;     &quot;chr2_KI270894v1_alt&quot;    
 [79] &quot;chr2_KN538362v1_fix&quot;     &quot;chr2_KN538363v1_fix&quot;    
 [81] &quot;chr2_KQ031384v1_fix&quot;     &quot;chr2_KQ983256v1_alt&quot;    
 [83] &quot;chr2_KZ208907v1_alt&quot;     &quot;chr2_KZ208908v1_alt&quot;    
 [85] &quot;chr3_GL000221v1_random&quot;  &quot;chr3_GL383526v1_alt&quot;    
 [87] &quot;chr3_JH636055v2_alt&quot;     &quot;chr3_KI270777v1_alt&quot;    
 [89] &quot;chr3_KI270778v1_alt&quot;     &quot;chr3_KI270779v1_alt&quot;    
 [91] &quot;chr3_KI270780v1_alt&quot;     &quot;chr3_KI270781v1_alt&quot;    
 [93] &quot;chr3_KI270782v1_alt&quot;     &quot;chr3_KI270783v1_alt&quot;    
 [95] &quot;chr3_KI270784v1_alt&quot;     &quot;chr3_KI270895v1_alt&quot;    
 [97] &quot;chr3_KI270924v1_alt&quot;     &quot;chr3_KI270934v1_alt&quot;    
 [99] &quot;chr3_KI270935v1_alt&quot;     &quot;chr3_KI270936v1_alt&quot;    
[101] &quot;chr3_KI270937v1_alt&quot;     &quot;chr3_KN196475v1_fix&quot;    
[103] &quot;chr3_KN196476v1_fix&quot;     &quot;chr3_KN538364v1_fix&quot;    
[105] &quot;chr3_KQ031385v1_fix&quot;     &quot;chr3_KQ031386v1_fix&quot;    
[107] &quot;chr3_KV766192v1_fix&quot;     &quot;chr3_KZ208909v1_alt&quot;    
[109] &quot;chr3_KZ559101v1_alt&quot;     &quot;chr3_KZ559102v1_alt&quot;    
[111] &quot;chr3_KZ559103v1_alt&quot;     &quot;chr3_KZ559104v1_fix&quot;    
[113] &quot;chr3_KZ559105v1_alt&quot;     &quot;chr4_GL000008v2_random&quot; 
[115] &quot;chr4_GL000257v2_alt&quot;     &quot;chr4_GL383527v1_alt&quot;    
[117] &quot;chr4_GL383528v1_alt&quot;     &quot;chr4_KI270785v1_alt&quot;    
[119] &quot;chr4_KI270786v1_alt&quot;     &quot;chr4_KI270787v1_alt&quot;    
[121] &quot;chr4_KI270788v1_alt&quot;     &quot;chr4_KI270789v1_alt&quot;    
[123] &quot;chr4_KI270790v1_alt&quot;     &quot;chr4_KI270896v1_alt&quot;    
[125] &quot;chr4_KI270925v1_alt&quot;     &quot;chr4_KQ090013v1_alt&quot;    
[127] &quot;chr4_KQ090014v1_alt&quot;     &quot;chr4_KQ090015v1_alt&quot;    
[129] &quot;chr4_KQ983257v1_fix&quot;     &quot;chr4_KQ983258v1_alt&quot;    
[131] &quot;chr4_KV766193v1_alt&quot;     &quot;chr5_GL000208v1_random&quot; 
[133] &quot;chr5_GL339449v2_alt&quot;     &quot;chr5_GL383530v1_alt&quot;    
[135] &quot;chr5_GL383531v1_alt&quot;     &quot;chr5_GL383532v1_alt&quot;    
[137] &quot;chr5_GL949742v1_alt&quot;     &quot;chr5_KI270791v1_alt&quot;    
[139] &quot;chr5_KI270792v1_alt&quot;     &quot;chr5_KI270793v1_alt&quot;    
[141] &quot;chr5_KI270794v1_alt&quot;     &quot;chr5_KI270795v1_alt&quot;    
[143] &quot;chr5_KI270796v1_alt&quot;     &quot;chr5_KI270897v1_alt&quot;    
[145] &quot;chr5_KI270898v1_alt&quot;     &quot;chr5_KN196477v1_alt&quot;    
[147] &quot;chr5_KV575243v1_alt&quot;     &quot;chr5_KV575244v1_fix&quot;    
[149] &quot;chr5_KZ208910v1_alt&quot;     &quot;chr6_GL000250v2_alt&quot;    
[151] &quot;chr6_GL000251v2_alt&quot;     &quot;chr6_GL000252v2_alt&quot;    
[153] &quot;chr6_GL000253v2_alt&quot;     &quot;chr6_GL000254v2_alt&quot;    
[155] &quot;chr6_GL000255v2_alt&quot;     &quot;chr6_GL000256v2_alt&quot;    
[157] &quot;chr6_GL383533v1_alt&quot;     &quot;chr6_KB021644v2_alt&quot;    
[159] &quot;chr6_KI270758v1_alt&quot;     &quot;chr6_KI270797v1_alt&quot;    
[161] &quot;chr6_KI270798v1_alt&quot;     &quot;chr6_KI270799v1_alt&quot;    
[163] &quot;chr6_KI270800v1_alt&quot;     &quot;chr6_KI270801v1_alt&quot;    
[165] &quot;chr6_KI270802v1_alt&quot;     &quot;chr6_KN196478v1_fix&quot;    
[167] &quot;chr6_KQ031387v1_fix&quot;     &quot;chr6_KQ090016v1_fix&quot;    
[169] &quot;chr6_KQ090017v1_alt&quot;     &quot;chr6_KV766194v1_fix&quot;    
[171] &quot;chr6_KZ208911v1_fix&quot;     &quot;chr7_GL383534v2_alt&quot;    
[173] &quot;chr7_KI270803v1_alt&quot;     &quot;chr7_KI270804v1_alt&quot;    
[175] &quot;chr7_KI270805v1_alt&quot;     &quot;chr7_KI270806v1_alt&quot;    
[177] &quot;chr7_KI270807v1_alt&quot;     &quot;chr7_KI270808v1_alt&quot;    
[179] &quot;chr7_KI270809v1_alt&quot;     &quot;chr7_KI270899v1_alt&quot;    
[181] &quot;chr7_KQ031388v1_fix&quot;     &quot;chr7_KV880764v1_fix&quot;    
[183] &quot;chr7_KV880765v1_fix&quot;     &quot;chr7_KZ208912v1_fix&quot;    
[185] &quot;chr7_KZ208913v1_alt&quot;     &quot;chr7_KZ559106v1_alt&quot;    
[187] &quot;chr8_KI270810v1_alt&quot;     &quot;chr8_KI270811v1_alt&quot;    
[189] &quot;chr8_KI270812v1_alt&quot;     &quot;chr8_KI270813v1_alt&quot;    
[191] &quot;chr8_KI270814v1_alt&quot;     &quot;chr8_KI270815v1_alt&quot;    
[193] &quot;chr8_KI270816v1_alt&quot;     &quot;chr8_KI270817v1_alt&quot;    
[195] &quot;chr8_KI270818v1_alt&quot;     &quot;chr8_KI270819v1_alt&quot;    
[197] &quot;chr8_KI270820v1_alt&quot;     &quot;chr8_KI270821v1_alt&quot;    
[199] &quot;chr8_KI270822v1_alt&quot;     &quot;chr8_KI270900v1_alt&quot;    
[201] &quot;chr8_KI270901v1_alt&quot;     &quot;chr8_KI270926v1_alt&quot;    
[203] &quot;chr8_KV880766v1_fix&quot;     &quot;chr8_KV880767v1_fix&quot;    
[205] &quot;chr8_KZ208914v1_fix&quot;     &quot;chr8_KZ208915v1_fix&quot;    
[207] &quot;chr8_KZ559107v1_alt&quot;     &quot;chr9_GL383539v1_alt&quot;    
[209] &quot;chr9_GL383540v1_alt&quot;     &quot;chr9_GL383541v1_alt&quot;    
[211] &quot;chr9_GL383542v1_alt&quot;     &quot;chr9_KI270717v1_random&quot; 
[213] &quot;chr9_KI270718v1_random&quot;  &quot;chr9_KI270719v1_random&quot; 
[215] &quot;chr9_KI270720v1_random&quot;  &quot;chr9_KI270823v1_alt&quot;    
[217] &quot;chr9_KN196479v1_fix&quot;     &quot;chr9_KQ090018v1_alt&quot;    
[219] &quot;chr9_KQ090019v1_alt&quot;     &quot;chr10_GL383545v1_alt&quot;   
[221] &quot;chr10_GL383546v1_alt&quot;    &quot;chr10_KI270824v1_alt&quot;   
[223] &quot;chr10_KI270825v1_alt&quot;    &quot;chr10_KN196480v1_fix&quot;   
[225] &quot;chr10_KN538365v1_fix&quot;    &quot;chr10_KN538366v1_fix&quot;   
[227] &quot;chr10_KN538367v1_fix&quot;    &quot;chr10_KQ090020v1_alt&quot;   
[229] &quot;chr10_KQ090021v1_fix&quot;    &quot;chr11_GL383547v1_alt&quot;   
[231] &quot;chr11_JH159136v1_alt&quot;    &quot;chr11_JH159137v1_alt&quot;   
[233] &quot;chr11_KI270721v1_random&quot; &quot;chr11_KI270826v1_alt&quot;   
[235] &quot;chr11_KI270827v1_alt&quot;    &quot;chr11_KI270829v1_alt&quot;   
[237] &quot;chr11_KI270830v1_alt&quot;    &quot;chr11_KI270831v1_alt&quot;   
[239] &quot;chr11_KI270832v1_alt&quot;    &quot;chr11_KI270902v1_alt&quot;   
[241] &quot;chr11_KI270903v1_alt&quot;    &quot;chr11_KI270927v1_alt&quot;   
[243] &quot;chr11_KN196481v1_fix&quot;    &quot;chr11_KN538368v1_alt&quot;   
[245] &quot;chr11_KQ090022v1_fix&quot;    &quot;chr11_KQ759759v1_fix&quot;   
[247] &quot;chr11_KV766195v1_fix&quot;    &quot;chr11_KZ559108v1_fix&quot;   
[249] &quot;chr11_KZ559109v1_fix&quot;    &quot;chr11_KZ559110v1_alt&quot;   
[251] &quot;chr11_KZ559111v1_alt&quot;    &quot;chr12_GL383549v1_alt&quot;   
[253] &quot;chr12_GL383550v2_alt&quot;    &quot;chr12_GL383551v1_alt&quot;   
[255] &quot;chr12_GL383552v1_alt&quot;    &quot;chr12_GL383553v2_alt&quot;   
[257] &quot;chr12_GL877875v1_alt&quot;    &quot;chr12_GL877876v1_alt&quot;   
[259] &quot;chr12_KI270833v1_alt&quot;    &quot;chr12_KI270834v1_alt&quot;   
[261] &quot;chr12_KI270835v1_alt&quot;    &quot;chr12_KI270836v1_alt&quot;   
[263] &quot;chr12_KI270837v1_alt&quot;    &quot;chr12_KI270904v1_alt&quot;   
[265] &quot;chr12_KN196482v1_fix&quot;    &quot;chr12_KN538369v1_fix&quot;   
[267] &quot;chr12_KN538370v1_fix&quot;    &quot;chr12_KQ090023v1_alt&quot;   
[269] &quot;chr12_KQ759760v1_fix&quot;    &quot;chr12_KZ208916v1_fix&quot;   
[271] &quot;chr12_KZ208917v1_fix&quot;    &quot;chr12_KZ208918v1_alt&quot;   
[273] &quot;chr12_KZ559112v1_alt&quot;    &quot;chr13_KI270838v1_alt&quot;   
[275] &quot;chr13_KI270839v1_alt&quot;    &quot;chr13_KI270840v1_alt&quot;   
[277] &quot;chr13_KI270841v1_alt&quot;    &quot;chr13_KI270842v1_alt&quot;   
[279] &quot;chr13_KI270843v1_alt&quot;    &quot;chr13_KN196483v1_fix&quot;   
[281] &quot;chr13_KN538371v1_fix&quot;    &quot;chr13_KN538372v1_fix&quot;   
[283] &quot;chr13_KN538373v1_fix&quot;    &quot;chr13_KQ090024v1_alt&quot;   
[285] &quot;chr13_KQ090025v1_alt&quot;    &quot;chr14_GL000009v2_random&quot;
[287] &quot;chr14_GL000194v1_random&quot; &quot;chr14_GL000225v1_random&quot;
[289] &quot;chr14_KI270722v1_random&quot; &quot;chr14_KI270723v1_random&quot;
[291] &quot;chr14_KI270724v1_random&quot; &quot;chr14_KI270725v1_random&quot;
[293] &quot;chr14_KI270726v1_random&quot; &quot;chr14_KI270844v1_alt&quot;   
[295] &quot;chr14_KI270845v1_alt&quot;    &quot;chr14_KI270846v1_alt&quot;   
[297] &quot;chr14_KI270847v1_alt&quot;    &quot;chr14_KZ208919v1_alt&quot;   
[299] &quot;chr14_KZ208920v1_fix&quot;    &quot;chr15_GL383554v1_alt&quot;   
[301] &quot;chr15_GL383555v2_alt&quot;    &quot;chr15_KI270727v1_random&quot;
[303] &quot;chr15_KI270848v1_alt&quot;    &quot;chr15_KI270849v1_alt&quot;   
[305] &quot;chr15_KI270850v1_alt&quot;    &quot;chr15_KI270851v1_alt&quot;   
[307] &quot;chr15_KI270852v1_alt&quot;    &quot;chr15_KI270905v1_alt&quot;   
[309] &quot;chr15_KI270906v1_alt&quot;    &quot;chr15_KN538374v1_fix&quot;   
[311] &quot;chr15_KQ031389v1_alt&quot;    &quot;chr16_GL383556v1_alt&quot;   
[313] &quot;chr16_GL383557v1_alt&quot;    &quot;chr16_KI270728v1_random&quot;
[315] &quot;chr16_KI270853v1_alt&quot;    &quot;chr16_KI270854v1_alt&quot;   
[317] &quot;chr16_KI270855v1_alt&quot;    &quot;chr16_KI270856v1_alt&quot;   
[319] &quot;chr16_KQ031390v1_alt&quot;    &quot;chr16_KQ090026v1_alt&quot;   
[321] &quot;chr16_KQ090027v1_alt&quot;    &quot;chr16_KV880768v1_fix&quot;   
[323] &quot;chr16_KZ208921v1_alt&quot;    &quot;chr16_KZ559113v1_fix&quot;   
[325] &quot;chr17_GL000205v2_random&quot; &quot;chr17_GL000258v2_alt&quot;   
[327] &quot;chr17_GL383563v3_alt&quot;    &quot;chr17_GL383564v2_alt&quot;   
[329] &quot;chr17_GL383565v1_alt&quot;    &quot;chr17_GL383566v1_alt&quot;   
[331] &quot;chr17_JH159146v1_alt&quot;    &quot;chr17_JH159147v1_alt&quot;   
[333] &quot;chr17_JH159148v1_alt&quot;    &quot;chr17_KI270729v1_random&quot;
[335] &quot;chr17_KI270730v1_random&quot; &quot;chr17_KI270857v1_alt&quot;   
[337] &quot;chr17_KI270858v1_alt&quot;    &quot;chr17_KI270859v1_alt&quot;   
[339] &quot;chr17_KI270860v1_alt&quot;    &quot;chr17_KI270861v1_alt&quot;   
[341] &quot;chr17_KI270862v1_alt&quot;    &quot;chr17_KI270907v1_alt&quot;   
[343] &quot;chr17_KI270908v1_alt&quot;    &quot;chr17_KI270909v1_alt&quot;   
[345] &quot;chr17_KI270910v1_alt&quot;    &quot;chr17_KV575245v1_fix&quot;   
[347] &quot;chr17_KV766196v1_fix&quot;    &quot;chr17_KV766197v1_alt&quot;   
[349] &quot;chr17_KV766198v1_alt&quot;    &quot;chr17_KZ559114v1_alt&quot;   
[351] &quot;chr18_GL383567v1_alt&quot;    &quot;chr18_GL383568v1_alt&quot;   
[353] &quot;chr18_GL383569v1_alt&quot;    &quot;chr18_GL383570v1_alt&quot;   
[355] &quot;chr18_GL383571v1_alt&quot;    &quot;chr18_GL383572v1_alt&quot;   
[357] &quot;chr18_KI270863v1_alt&quot;    &quot;chr18_KI270864v1_alt&quot;   
[359] &quot;chr18_KI270911v1_alt&quot;    &quot;chr18_KI270912v1_alt&quot;   
[361] &quot;chr18_KQ090028v1_fix&quot;    &quot;chr18_KQ458385v1_alt&quot;   
[363] &quot;chr18_KZ208922v1_fix&quot;    &quot;chr18_KZ559115v1_fix&quot;   
[365] &quot;chr18_KZ559116v1_alt&quot;    &quot;chr19_GL000209v2_alt&quot;   
[367] &quot;chr19_GL383573v1_alt&quot;    &quot;chr19_GL383574v1_alt&quot;   
[369] &quot;chr19_GL383575v2_alt&quot;    &quot;chr19_GL383576v1_alt&quot;   
[371] &quot;chr19_GL949746v1_alt&quot;    &quot;chr19_GL949747v2_alt&quot;   
[373] &quot;chr19_GL949748v2_alt&quot;    &quot;chr19_GL949749v2_alt&quot;   
[375] &quot;chr19_GL949750v2_alt&quot;    &quot;chr19_GL949751v2_alt&quot;   
[377] &quot;chr19_GL949752v1_alt&quot;    &quot;chr19_GL949753v2_alt&quot;   
[379] &quot;chr19_KI270865v1_alt&quot;    &quot;chr19_KI270866v1_alt&quot;   
[381] &quot;chr19_KI270867v1_alt&quot;    &quot;chr19_KI270868v1_alt&quot;   
[383] &quot;chr19_KI270882v1_alt&quot;    &quot;chr19_KI270883v1_alt&quot;   
[385] &quot;chr19_KI270884v1_alt&quot;    &quot;chr19_KI270885v1_alt&quot;   
[387] &quot;chr19_KI270886v1_alt&quot;    &quot;chr19_KI270887v1_alt&quot;   
[389] &quot;chr19_KI270888v1_alt&quot;    &quot;chr19_KI270889v1_alt&quot;   
[391] &quot;chr19_KI270890v1_alt&quot;    &quot;chr19_KI270891v1_alt&quot;   
[393] &quot;chr19_KI270914v1_alt&quot;    &quot;chr19_KI270915v1_alt&quot;   
[395] &quot;chr19_KI270916v1_alt&quot;    &quot;chr19_KI270917v1_alt&quot;   
[397] &quot;chr19_KI270918v1_alt&quot;    &quot;chr19_KI270919v1_alt&quot;   
[399] &quot;chr19_KI270920v1_alt&quot;    &quot;chr19_KI270921v1_alt&quot;   
[401] &quot;chr19_KI270922v1_alt&quot;    &quot;chr19_KI270923v1_alt&quot;   
[403] &quot;chr19_KI270929v1_alt&quot;    &quot;chr19_KI270930v1_alt&quot;   
[405] &quot;chr19_KI270931v1_alt&quot;    &quot;chr19_KI270932v1_alt&quot;   
[407] &quot;chr19_KI270933v1_alt&quot;    &quot;chr19_KI270938v1_alt&quot;   
[409] &quot;chr19_KN196484v1_fix&quot;    &quot;chr19_KQ458386v1_fix&quot;   
[411] &quot;chr19_KV575246v1_alt&quot;    &quot;chr19_KV575247v1_alt&quot;   
[413] &quot;chr19_KV575248v1_alt&quot;    &quot;chr19_KV575249v1_alt&quot;   
[415] &quot;chr19_KV575250v1_alt&quot;    &quot;chr19_KV575251v1_alt&quot;   
[417] &quot;chr19_KV575252v1_alt&quot;    &quot;chr19_KV575253v1_alt&quot;   
[419] &quot;chr19_KV575254v1_alt&quot;    &quot;chr19_KV575255v1_alt&quot;   
[421] &quot;chr19_KV575256v1_alt&quot;    &quot;chr19_KV575257v1_alt&quot;   
[423] &quot;chr19_KV575258v1_alt&quot;    &quot;chr19_KV575259v1_alt&quot;   
[425] &quot;chr19_KV575260v1_alt&quot;    &quot;chr20_GL383577v2_alt&quot;   
[427] &quot;chr20_KI270869v1_alt&quot;    &quot;chr20_KI270870v1_alt&quot;   
[429] &quot;chr20_KI270871v1_alt&quot;    &quot;chr21_GL383578v2_alt&quot;   
[431] &quot;chr21_GL383579v2_alt&quot;    &quot;chr21_GL383580v2_alt&quot;   
[433] &quot;chr21_GL383581v2_alt&quot;    &quot;chr21_KI270872v1_alt&quot;   
[435] &quot;chr21_KI270873v1_alt&quot;    &quot;chr21_KI270874v1_alt&quot;   
[437] &quot;chr22_GL383582v2_alt&quot;    &quot;chr22_GL383583v2_alt&quot;   
[439] &quot;chr22_KB663609v1_alt&quot;    &quot;chr22_KI270731v1_random&quot;
[441] &quot;chr22_KI270732v1_random&quot; &quot;chr22_KI270733v1_random&quot;
[443] &quot;chr22_KI270734v1_random&quot; &quot;chr22_KI270735v1_random&quot;
[445] &quot;chr22_KI270736v1_random&quot; &quot;chr22_KI270737v1_random&quot;
[447] &quot;chr22_KI270738v1_random&quot; &quot;chr22_KI270739v1_random&quot;
[449] &quot;chr22_KI270875v1_alt&quot;    &quot;chr22_KI270876v1_alt&quot;   
[451] &quot;chr22_KI270877v1_alt&quot;    &quot;chr22_KI270878v1_alt&quot;   
[453] &quot;chr22_KI270879v1_alt&quot;    &quot;chr22_KI270928v1_alt&quot;   
[455] &quot;chr22_KN196485v1_alt&quot;    &quot;chr22_KN196486v1_alt&quot;   
[457] &quot;chr22_KQ458387v1_alt&quot;    &quot;chr22_KQ458388v1_alt&quot;   
[459] &quot;chr22_KQ759761v1_alt&quot;    &quot;chr22_KQ759762v1_fix&quot;   
[461] &quot;chrX_KI270880v1_alt&quot;     &quot;chrX_KI270881v1_alt&quot;    
[463] &quot;chrX_KI270913v1_alt&quot;     &quot;chrX_KV766199v1_alt&quot;    
[465] &quot;chrY_KI270740v1_random&quot;  &quot;chrY_KN196487v1_fix&quot;    
[467] &quot;chrY_KZ208923v1_fix&quot;     &quot;chrY_KZ208924v1_fix&quot;    
[469] &quot;chrUn_GL000195v1&quot;        &quot;chrUn_GL000213v1&quot;       
[471] &quot;chrUn_GL000214v1&quot;        &quot;chrUn_GL000216v2&quot;       
[473] &quot;chrUn_GL000218v1&quot;        &quot;chrUn_GL000219v1&quot;       
[475] &quot;chrUn_GL000220v1&quot;        &quot;chrUn_GL000224v1&quot;       
[477] &quot;chrUn_GL000226v1&quot;        &quot;chrUn_KI270302v1&quot;       
[479] &quot;chrUn_KI270303v1&quot;        &quot;chrUn_KI270304v1&quot;       
[481] &quot;chrUn_KI270305v1&quot;        &quot;chrUn_KI270310v1&quot;       
[483] &quot;chrUn_KI270311v1&quot;        &quot;chrUn_KI270312v1&quot;       
[485] &quot;chrUn_KI270315v1&quot;        &quot;chrUn_KI270316v1&quot;       
[487] &quot;chrUn_KI270317v1&quot;        &quot;chrUn_KI270320v1&quot;       
[489] &quot;chrUn_KI270322v1&quot;        &quot;chrUn_KI270329v1&quot;       
[491] &quot;chrUn_KI270330v1&quot;        &quot;chrUn_KI270333v1&quot;       
[493] &quot;chrUn_KI270334v1&quot;        &quot;chrUn_KI270335v1&quot;       
[495] &quot;chrUn_KI270336v1&quot;        &quot;chrUn_KI270337v1&quot;       
[497] &quot;chrUn_KI270338v1&quot;        &quot;chrUn_KI270340v1&quot;       
[499] &quot;chrUn_KI270362v1&quot;        &quot;chrUn_KI270363v1&quot;       
[501] &quot;chrUn_KI270364v1&quot;        &quot;chrUn_KI270366v1&quot;       
[503] &quot;chrUn_KI270371v1&quot;        &quot;chrUn_KI270372v1&quot;       
[505] &quot;chrUn_KI270373v1&quot;        &quot;chrUn_KI270374v1&quot;       
[507] &quot;chrUn_KI270375v1&quot;        &quot;chrUn_KI270376v1&quot;       
[509] &quot;chrUn_KI270378v1&quot;        &quot;chrUn_KI270379v1&quot;       
[511] &quot;chrUn_KI270381v1&quot;        &quot;chrUn_KI270382v1&quot;       
[513] &quot;chrUn_KI270383v1&quot;        &quot;chrUn_KI270384v1&quot;       
[515] &quot;chrUn_KI270385v1&quot;        &quot;chrUn_KI270386v1&quot;       
[517] &quot;chrUn_KI270387v1&quot;        &quot;chrUn_KI270388v1&quot;       
[519] &quot;chrUn_KI270389v1&quot;        &quot;chrUn_KI270390v1&quot;       
[521] &quot;chrUn_KI270391v1&quot;        &quot;chrUn_KI270392v1&quot;       
[523] &quot;chrUn_KI270393v1&quot;        &quot;chrUn_KI270394v1&quot;       
[525] &quot;chrUn_KI270395v1&quot;        &quot;chrUn_KI270396v1&quot;       
[527] &quot;chrUn_KI270411v1&quot;        &quot;chrUn_KI270412v1&quot;       
[529] &quot;chrUn_KI270414v1&quot;        &quot;chrUn_KI270417v1&quot;       
[531] &quot;chrUn_KI270418v1&quot;        &quot;chrUn_KI270419v1&quot;       
[533] &quot;chrUn_KI270420v1&quot;        &quot;chrUn_KI270422v1&quot;       
[535] &quot;chrUn_KI270423v1&quot;        &quot;chrUn_KI270424v1&quot;       
[537] &quot;chrUn_KI270425v1&quot;        &quot;chrUn_KI270429v1&quot;       
[539] &quot;chrUn_KI270435v1&quot;        &quot;chrUn_KI270438v1&quot;       
[541] &quot;chrUn_KI270442v1&quot;        &quot;chrUn_KI270448v1&quot;       
[543] &quot;chrUn_KI270465v1&quot;        &quot;chrUn_KI270466v1&quot;       
[545] &quot;chrUn_KI270467v1&quot;        &quot;chrUn_KI270468v1&quot;       
[547] &quot;chrUn_KI270507v1&quot;        &quot;chrUn_KI270508v1&quot;       
[549] &quot;chrUn_KI270509v1&quot;        &quot;chrUn_KI270510v1&quot;       
[551] &quot;chrUn_KI270511v1&quot;        &quot;chrUn_KI270512v1&quot;       
[553] &quot;chrUn_KI270515v1&quot;        &quot;chrUn_KI270516v1&quot;       
[555] &quot;chrUn_KI270517v1&quot;        &quot;chrUn_KI270518v1&quot;       
[557] &quot;chrUn_KI270519v1&quot;        &quot;chrUn_KI270521v1&quot;       
[559] &quot;chrUn_KI270522v1&quot;        &quot;chrUn_KI270528v1&quot;       
[561] &quot;chrUn_KI270529v1&quot;        &quot;chrUn_KI270530v1&quot;       
[563] &quot;chrUn_KI270538v1&quot;        &quot;chrUn_KI270539v1&quot;       
[565] &quot;chrUn_KI270544v1&quot;        &quot;chrUn_KI270548v1&quot;       
[567] &quot;chrUn_KI270579v1&quot;        &quot;chrUn_KI270580v1&quot;       
[569] &quot;chrUn_KI270581v1&quot;        &quot;chrUn_KI270582v1&quot;       
[571] &quot;chrUn_KI270583v1&quot;        &quot;chrUn_KI270584v1&quot;       
[573] &quot;chrUn_KI270587v1&quot;        &quot;chrUn_KI270588v1&quot;       
[575] &quot;chrUn_KI270589v1&quot;        &quot;chrUn_KI270590v1&quot;       
[577] &quot;chrUn_KI270591v1&quot;        &quot;chrUn_KI270593v1&quot;       
[579] &quot;chrUn_KI270741v1&quot;        &quot;chrUn_KI270742v1&quot;       
[581] &quot;chrUn_KI270743v1&quot;        &quot;chrUn_KI270744v1&quot;       
[583] &quot;chrUn_KI270745v1&quot;        &quot;chrUn_KI270746v1&quot;       
[585] &quot;chrUn_KI270747v1&quot;        &quot;chrUn_KI270748v1&quot;       
[587] &quot;chrUn_KI270749v1&quot;        &quot;chrUn_KI270750v1&quot;       
[589] &quot;chrUn_KI270751v1&quot;        &quot;chrUn_KI270752v1&quot;       
[591] &quot;chrUn_KI270753v1&quot;        &quot;chrUn_KI270754v1&quot;       
[593] &quot;chrUn_KI270755v1&quot;        &quot;chrUn_KI270756v1&quot;       
[595] &quot;chrUn_KI270757v1&quot;       </code></pre>
<pre><code> [1] &quot;chr1&quot;  &quot;chr2&quot;  &quot;chr3&quot;  &quot;chr4&quot;  &quot;chr5&quot;  &quot;chr6&quot;  &quot;chr7&quot;  &quot;chr8&quot;  &quot;chr9&quot; 
[10] &quot;chr10&quot; &quot;chr11&quot; &quot;chr12&quot; &quot;chr13&quot; &quot;chr14&quot; &quot;chr15&quot; &quot;chr16&quot; &quot;chr17&quot; &quot;chr18&quot;
[19] &quot;chr19&quot; &quot;chr20&quot; &quot;chr21&quot; &quot;chr22&quot; &quot;chrX&quot;  &quot;chrY&quot; </code></pre>
<pre><code>GRanges object with 4808 ranges and 14 metadata columns:
         seqnames            ranges strand |     logFC    logCPM        LR
            &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt; &lt;numeric&gt;
  339451     chr1     960584-965719      + |  1.444466   1.13402  12.01408
  375790     chr1   1020120-1056118      + |  1.428524   4.58160  16.37063
   54991     chr1   1081818-1116361      - |  1.179575   2.41598  11.76935
   51150     chr1   1216909-1232067      - |  0.452477   5.35276   7.51441
    6339     chr1   1280436-1292029      + |  1.051756   2.24617   8.46603
     ...      ...               ...    ... .       ...       ...       ...
    8888    chr21 46235133-46286297      - |  0.658232   6.40840   18.0945
   54058    chr21 46300181-46323875      - |  1.744116   1.84115   24.2072
    5116    chr21 46324141-46445769      + |  0.772067   6.81541   25.6821
   23181    chr21 46458891-46570015      + |  0.921378   6.25343   33.9688
    3275    chr21 46635595-46665124      + |  0.485232   6.94981   11.5717
              PValue         FDR         ENSEMBL    ENTREZID      SYMBOL
           &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt; &lt;character&gt;
  339451 5.28001e-04 0.003696271 ENSG00000187961      339451      KLHL17
  375790 5.20862e-05 0.000684321 ENSG00000188157      375790        AGRN
   54991 6.02139e-04 0.004066014 ENSG00000131591       54991    C1orf159
   51150 6.12073e-03 0.023149708 ENSG00000078808       51150        SDF4
    6339 3.61840e-03 0.015531915 ENSG00000162572        6339      SCNN1D
     ...         ...         ...             ...         ...         ...
    8888 2.10209e-05 3.49228e-04 ENSG00000160294        8888      MCM3AP
   54058 8.65097e-07 3.45406e-05 ENSG00000160298       54058    C21orf58
    5116 4.02551e-07 1.96724e-05 ENSG00000160299        5116        PCNT
   23181 5.60024e-09 7.80794e-07 ENSG00000160305       23181       DIP2A
    3275 6.69616e-04 4.42441e-03 ENSG00000160310        3275       PRMT2
                       GENENAME         CHR     UNIPROT       ALIAS     score
                    &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;numeric&gt;
  339451 kelch like family me..           1      Q6TDP4          AF   2.43224
  375790                  agrin           1      O00468       AGRIN   3.16474
   54991 chromosome 1 open re..           1      Q96HA4    C1orf159   2.39083
   51150 stromal cell derived..           1  A0A024R084       Cab45   1.63545
    6339 sodium channel epith..           1      A6NNF7       ENaCd   1.80877
     ...                    ...         ...         ...         ...       ...
    8888 minichromosome maint..          21      O60318        GANP   3.45689
   54058 chromosome 21 open r..          21      P58505    C21orf58   4.46167
    5116            pericentrin          21      O95613         KEN   4.70614
   23181 disco interacting pr..          21      Q14689   C21orf106   6.10746
    3275 protein arginine met..          21      P55345     HRMT1L1   2.35414
             itemRgb
         &lt;character&gt;
  339451     #AA0055
  375790     #AA0055
   54991     #8D0071
   51150     #71008D
    6339     #8D0071
     ...         ...
    8888     #8D0071
   54058     #AA0055
    5116     #8D0071
   23181     #8D0071
    3275     #71008D
  -------
  seqinfo: 24 sequences from hg38 genome</code></pre>
<pre><code>using coord:genome to parse x scale</code></pre>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-12-1.png" width="672" style="display: block; margin: auto;" /></p>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<pre><code>Scale for &#39;x&#39; is already present. Adding another scale for &#39;x&#39;, which will
replace the existing scale.</code></pre>
<p><img src="figure/differential_expression_analysis.Rmd/unnamed-chunk-12-2.png" width="672" style="display: block; margin: auto;" /></p>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] circlize_0.4.14                         
 [2] ComplexHeatmap_2.8.0                    
 [3] pathview_1.32.0                         
 [4] forcats_0.5.1                           
 [5] stringr_1.4.0                           
 [6] purrr_0.3.4                             
 [7] readr_2.1.2                             
 [8] tidyr_1.2.0                             
 [9] tibble_3.1.6                            
[10] tidyverse_1.3.1                         
[11] karyoploteR_1.18.0                      
[12] regioneR_1.24.0                         
[13] ggridges_0.5.3                          
[14] enrichplot_1.12.3                       
[15] clusterProfiler_4.0.5                   
[16] ggbio_1.40.0                            
[17] rtracklayer_1.52.1                      
[18] TxDb.Hsapiens.UCSC.hg38.knownGene_3.13.0
[19] GenomicFeatures_1.44.2                  
[20] GenomicRanges_1.44.0                    
[21] GenomeInfoDb_1.28.4                     
[22] pcaExplorer_2.18.0                      
[23] org.Hs.eg.db_3.13.0                     
[24] AnnotationDbi_1.54.1                    
[25] IRanges_2.28.0                          
[26] S4Vectors_0.32.3                        
[27] Biobase_2.54.0                          
[28] BiocGenerics_0.40.0                     
[29] DBI_1.1.2                               
[30] ggrepel_0.9.1                           
[31] openxlsx_4.2.5                          
[32] edgeR_3.34.1                            
[33] limma_3.50.1                            
[34] dplyr_1.0.8                             
[35] ggplot2_3.3.5                           
[36] workflowr_1.7.0                         

loaded via a namespace (and not attached):
  [1] rsvd_1.0.5                  Hmisc_4.6-0                
  [3] ps_1.6.0                    Rsamtools_2.8.0            
  [5] foreach_1.5.2               rprojroot_2.0.2            
  [7] crayon_1.5.0                MASS_7.3-55                
  [9] nlme_3.1-155                backports_1.4.1            
 [11] reprex_2.0.1                GOSemSim_2.18.1            
 [13] rlang_1.0.2                 XVector_0.32.0             
 [15] readxl_1.3.1                irlba_2.3.5                
 [17] SparseM_1.81                callr_3.7.0                
 [19] filelock_1.0.2              GOstats_2.58.0             
 [21] BiocParallel_1.28.3         rjson_0.2.21               
 [23] bit64_4.0.5                 glue_1.6.2                 
 [25] pheatmap_1.0.12             rngtools_1.5.2             
 [27] parallel_4.1.0              processx_3.5.2             
 [29] shinyAce_0.4.1              shinydashboard_0.7.2       
 [31] DOSE_3.18.3                 haven_2.4.3                
 [33] tidyselect_1.1.2            SummarizedExperiment_1.22.0
 [35] XML_3.99-0.9                GenomicAlignments_1.28.0   
 [37] xtable_1.8-4                magrittr_2.0.2             
 [39] evaluate_0.15               cli_3.2.0                  
 [41] zlibbioc_1.40.0             rstudioapi_0.13            
 [43] whisker_0.4                 bslib_0.3.1                
 [45] rpart_4.1.16                fastmatch_1.1-3            
 [47] ensembldb_2.16.4            treeio_1.16.2              
 [49] shiny_1.7.1                 BiocSingular_1.8.1         
 [51] xfun_0.30                   clue_0.3-60                
 [53] cluster_2.1.2               tidygraph_1.2.0            
 [55] TSP_1.2-0                   KEGGREST_1.32.0            
 [57] biovizBase_1.40.0           threejs_0.3.3              
 [59] ape_5.6-2                   dendextend_1.15.2          
 [61] Biostrings_2.60.2           png_0.1-7                  
 [63] reshape_0.8.8               withr_2.5.0                
 [65] shinyBS_0.61                bitops_1.0-7               
 [67] ggforce_0.3.3               RBGL_1.68.0                
 [69] plyr_1.8.6                  cellranger_1.1.0           
 [71] GSEABase_1.54.0             AnnotationFilter_1.16.0    
 [73] dqrng_0.3.0                 pillar_1.7.0               
 [75] GlobalOptions_0.1.2         cachem_1.0.6               
 [77] fs_1.5.2                    GetoptLong_1.0.5           
 [79] DelayedMatrixStats_1.14.3   vctrs_0.3.8                
 [81] ellipsis_0.3.2              generics_0.1.2             
 [83] NMF_0.23.0                  tools_4.1.0                
 [85] foreign_0.8-82              munsell_0.5.0              
 [87] tweenr_1.0.2                fgsea_1.18.0               
 [89] DelayedArray_0.18.0         fastmap_1.1.0              
 [91] compiler_4.1.0              httpuv_1.6.5               
 [93] pkgmaker_0.32.2             plotly_4.10.0              
 [95] GenomeInfoDbData_1.2.6      gridExtra_2.3              
 [97] lattice_0.20-45             AnnotationForge_1.34.1     
 [99] utf8_1.2.2                  later_1.3.0                
[101] BiocFileCache_2.0.0         jsonlite_1.8.0             
[103] GGally_2.1.2                scales_1.1.1               
[105] ScaledMatrix_1.0.0          graph_1.70.0               
[107] sparseMatrixStats_1.4.2     tidytree_0.3.9             
[109] genefilter_1.74.1           lazyeval_0.2.2             
[111] promises_1.2.0.1            doParallel_1.0.17          
[113] latticeExtra_0.6-29         checkmate_2.0.0            
[115] rmarkdown_2.13              cowplot_1.1.1              
[117] webshot_0.5.2               dichromat_2.0-0            
[119] downloader_0.4              BSgenome_1.60.0            
[121] igraph_1.2.11               survival_3.3-1             
[123] yaml_2.3.5                  htmltools_0.5.2            
[125] memoise_2.0.1               VariantAnnotation_1.38.0   
[127] BiocIO_1.2.0                locfit_1.5-9.5             
[129] seriation_1.3.3             PCAtools_2.4.0             
[131] graphlayouts_0.8.0          viridisLite_0.4.0          
[133] digest_0.6.29               assertthat_0.2.1           
[135] mime_0.12                   rappdirs_0.3.3             
[137] registry_0.5-1              RSQLite_2.2.10             
[139] yulab.utils_0.0.4           data.table_1.14.2          
[141] blob_1.2.2                  labeling_0.4.2             
[143] splines_4.1.0               Formula_1.2-4              
[145] Cairo_1.5-15                OrganismDbi_1.34.0         
[147] ProtGenerics_1.26.0         RCurl_1.98-1.6             
[149] broom_0.7.12                hms_1.1.1                  
[151] modelr_0.1.8                colorspace_2.0-3           
[153] base64enc_0.1-3             BiocManager_1.30.16        
[155] shape_1.4.6                 aplot_0.1.2                
[157] nnet_7.3-17                 sass_0.4.0                 
[159] Rcpp_1.0.8.3                fansi_1.0.2                
[161] tzdb_0.2.0                  R6_2.5.1                   
[163] lifecycle_1.0.1             zip_2.2.0                  
[165] curl_4.3.2                  jquerylib_0.1.4            
[167] DO.db_2.9                   Matrix_1.4-0               
[169] qvalue_2.24.0               RColorBrewer_1.1-2         
[171] iterators_1.0.14            topGO_2.44.0               
[173] htmlwidgets_1.5.4           bamsignals_1.24.0          
[175] beachmat_2.8.1              polyclip_1.10-0            
[177] biomaRt_2.48.3              crosstalk_1.2.0            
[179] shadowtext_0.1.1            gridGraphics_0.5-1         
[181] rvest_1.0.2                 htmlTable_2.4.0            
[183] patchwork_1.1.1             KEGGgraph_1.52.0           
[185] codetools_0.2-18            matrixStats_0.61.0         
[187] lubridate_1.8.0             GO.db_3.13.0               
[189] getPass_0.2-2               prettyunits_1.1.1          
[191] dbplyr_2.1.1                gridBase_0.4-7             
[193] gtable_0.3.0                git2r_0.30.1               
[195] highr_0.9                   ggfun_0.0.5                
[197] httr_1.4.2                  stringi_1.7.6              
[199] progress_1.2.2              reshape2_1.4.4             
[201] farver_2.1.0                heatmaply_1.3.0            
[203] annotate_1.70.0             viridis_0.6.2              
[205] Rgraphviz_2.36.0            ggtree_3.0.4               
[207] DT_0.21                     xml2_1.3.3                 
[209] bezier_1.1.2                restfulr_0.0.13            
[211] geneplotter_1.70.0          ggplotify_0.1.0            
[213] Category_2.58.0             DESeq2_1.32.0              
[215] bit_4.0.4                   scatterpie_0.1.7           
[217] jpeg_0.1-9                  MatrixGenerics_1.4.3       
[219] ggraph_2.0.5                pkgconfig_2.0.3            
[221] knitr_1.37                 </code></pre>
</div>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
