<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="neurodevdisorder" />

<meta name="date" content="2022-09-27" />

<title>Figure1</title>

<script src="site_libs/header-attrs-2.16/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/spacelab.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/main/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark the anchor link active (and if it's in a dropdown, also mark that active)
  var dropdown = menuAnchor.closest('li.dropdown');
  if (window.bootstrap) { // Bootstrap 4+
    menuAnchor.addClass('active');
    dropdown.find('> .dropdown-toggle').addClass('active');
  } else { // Bootstrap 3
    menuAnchor.parent().addClass('active');
    dropdown.addClass('active');
  }

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">DS-Hs-Multi-omics</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class=" https://img.icons8.com/ios/250/000000/home.png"></span>
     
    Home
  </a>
</li>
<li>
  <a href="about.html">
    <span class=" https://img.icons8.com/ios/250/000000/info.png"></span>
     
    About
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="https://img.icons8.com/ios/250/000000/combo https://img.icons8.com/ios/250/000000/combo-chart.png"></span>
     
    Analyses
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li class="dropdown-header">Gene</li>
    <li>
      <a href="differential_expression_analysis.html">Run differential gene expression analysis</a>
    </li>
    <li class="divider"></li>
    <li class="dropdown-header">Transcript</li>
  </ul>
</li>
<li>
  <a href="shiny-link.html">
    <span class="fa fa-bar-chart"></span>
     
    Explore Data
  </a>
</li>
<li>
  <a href="license.html">License</a>
</li>
<li>
  <a href="https://www.iit.it/people-details/-/people/laura-cancedda">
    <span class="https://img.icons8.com/ios/250/000000/test https://img.icons8.com/ios/250/000000/test-tube.png"></span>
     
  </a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/neurodevdisorder/HumanMultiomics">
    <span class="fab fa-github"></span>
     
    Source code
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">Figure1</h1>
<h4 class="author">neurodevdisorder</h4>
<h4 class="date">September 27, 2022</h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span>
workflowr <span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2022-09-27
</p>
<p>
<strong>Checks:</strong> <span
class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7
<span class="glyphicon glyphicon-exclamation-sign text-danger"
aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>HumanMultiomics/</code> <span
class="glyphicon glyphicon-question-sign" aria-hidden="true"
title="This is the local directory in which the code in this file was executed.">
</span>
</p>
<p>
This reproducible <a href="https://rmarkdown.rstudio.com">R Markdown</a>
analysis was created with <a
  href="https://github.com/workflowr/workflowr">workflowr</a> (version
1.7.0). The <em>Checks</em> tab describes the reproducibility checks
that were applied when the results were created. The <em>Past
versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date
</a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git
repository, you know the exact version of the code that produced these
results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the
global environment can affect the analysis in your R Markdown file in
unknown ways. For reproduciblity it’s best to always run the code in an
empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20220715code">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Seed:</strong>
<code>set.seed(20220715)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20220715code"
class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20220715)</code> was run prior to running
the code in the R Markdown file. Setting a seed ensures that any results
that rely on randomness, e.g. subsampling or permutations, are
reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Session information:</strong>
recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded"
class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package
versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be
confident that you successfully produced the results during this
run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr
project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrongahrefhttpsgithubcomneurodevdisorderHumanMultiomicstree528e160a283cc78d4c04218b089644e2c2cff44btargetblank528e160a">
<span class="glyphicon glyphicon-ok text-success"
aria-hidden="true"></span> <strong>Repository version:</strong>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/tree/528e160a283cc78d4c04218b089644e2c2cff44b" target="_blank">528e160</a>
</a>
</p>
</div>
<div
id="strongRepositoryversionstrongahrefhttpsgithubcomneurodevdisorderHumanMultiomicstree528e160a283cc78d4c04218b089644e2c2cff44btargetblank528e160a"
class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development
and connecting the code version to the results is critical for
reproducibility.
</p>
<p>
The results in this page were generated with repository version
<a href="https://github.com/neurodevdisorder/HumanMultiomics/tree/528e160a283cc78d4c04218b089644e2c2cff44b" target="_blank">528e160</a>.
See the <em>Past versions</em> tab to see a history of the changes made
to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for
the analysis have been committed to Git prior to generating the results
(you can use <code>wflow_publish</code> or
<code>wflow_git_commit</code>). workflowr only checks the R Markdown
file, but you know if there are other scripts or data files that it
depends on. Below is the status of the Git repository when the results
were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .DS_Store
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/.DS_Store
    Ignored:    analysis/.Rhistory
    Ignored:    data/.DS_Store
    Ignored:    figure/.DS_Store

Untracked files:
    Untracked:  output/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not
included in this status report because it is ok for generated content to
have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were
made to the R Markdown (<code>analysis/Figure1.Rmd</code>) and HTML
(<code>docs/Figure1.html</code>) files. If you’ve configured a remote
Git repository (see <code>?wflow_git_remote</code>), click on the
hyperlinks in the table below to view the files as they were in that
past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/6f1f98020af2cb3be75e50b53d2f0340667a241c/analysis/Figure1.Rmd" target="_blank">6f1f980</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
<td>
first commit
</td>
</tr>
<tr>
<td>
html
</td>
<td>
<a href="https://rawcdn.githack.com/neurodevdisorder/HumanMultiomics/6f1f98020af2cb3be75e50b53d2f0340667a241c/docs/Figure1.html" target="_blank">6f1f980</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
<td>
first commit
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<div id="introduction" class="section level2">
<h2>Introduction</h2>
<p>These commands were used to generate different panels of Figure
1.</p>
<p>Figure 1A is designed in Adobe Illustrator and Figure 1B, Supp.
Figure 1D are prepared with the help of venny2.1 (<a
href="https://bioinfogp.cnb.csic.es/tools/venny/"
class="uri">https://bioinfogp.cnb.csic.es/tools/venny/</a>)</p>
<p>First, we will load all the packages required to perform this
analysis</p>
<pre class="r"><code>library(ggplot2)
library(dplyr)
library(edgeR)
library(openxlsx)
library(ggrepel)
library(DBI)
library(org.Hs.eg.db)
library(&quot;pcaExplorer&quot;)
library(TxDb.Hsapiens.UCSC.hg38.knownGene)
library(rtracklayer)
library(ggbio)
library(clusterProfiler)
library(enrichplot)
library(ggridges)
library(karyoploteR)
library(tidyverse)
library(forcats)
library(pathview)
library(ComplexHeatmap)
library(circlize)
library(rsample)
library(magrittr)
library(superheat)</code></pre>
<div id="figure-1c" class="section level4">
<h4>Figure 1C</h4>
<div id="hippocampus" class="section level5">
<h5>Hippocampus</h5>
<pre class="r"><code>diff_gene_transcript_hippo&lt;- read.table(file= &quot;data/diff_gene_transcript_hippo.txt&quot;, header=T, row.names=1)
head(diff_gene_transcript_hippo)</code></pre>
<pre><code>                    LogFC Type
ENSG00000127507 1.6532658 Gene
ENSG00000148671 0.8430004 Gene
ENSG00000204172 1.0022032 Gene
ENSG00000197976 0.5414043 Gene
ENSG00000128394 1.0469862 Gene
ENSG00000160072 0.9828805 Gene</code></pre>
<pre class="r"><code>diff_gene_transcript_hippo_density_plot&lt;- ggplot(diff_gene_transcript_hippo, aes(LogFC))+
  geom_density(aes(fill=factor(Type)), alpha=0.8) +
  scale_fill_manual(values = c(&quot;#AA7DCE&quot;,&quot;#FEEAFA&quot;)) +
  xlim(-14,14)+
  ylim(0,0.9)+
  labs(x=&quot;LogFC&quot;,
       fill=&quot;#Type&quot;)+theme_bw()+
  theme(text=element_text(size=14),axis.text.x = element_text(color = &quot;black&quot;),axis.text.y = element_text(color=&quot;black&quot;),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())
diff_gene_transcript_hippo_density_plot</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-1-1">
Past versions of unnamed-chunk-1-1.png
</button>
</p>
<div id="fig-unnamed-chunk-1-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-1-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="cortex" class="section level5">
<h5>Cortex</h5>
<pre class="r"><code>diff_gene_transcript_cortex&lt;- read.table(file= &quot;data/diff_gene_transcript_cortex.txt&quot;, header=T, row.names=1)
head(diff_gene_transcript_cortex)</code></pre>
<pre><code>                    LogFC Type
ENSG00000128274 2.0452885 Gene
ENSG00000181409 0.9393429 Gene
ENSG00000165029 0.8550180 Gene
ENSG00000107331 0.9181746 Gene
ENSG00000064687 0.9366807 Gene
ENSG00000197150 0.8312631 Gene</code></pre>
<pre class="r"><code>diff_gene_transcript_cortex_density_plot&lt;- ggplot(diff_gene_transcript_cortex, aes(LogFC))+
  geom_density(aes(fill=factor(Type)), alpha=0.8) +
  scale_fill_manual(values = c(&quot;#F4A5AE&quot;,&quot;#F7E5DB&quot;)) +
  xlim(-14,14)+
  ylim(0,0.9)+
  labs(x=&quot;LogFC&quot;,
       fill=&quot;#Type&quot;)+theme_bw()+
  theme(text=element_text(size=14),axis.text.x = element_text(color = &quot;black&quot;),axis.text.y = element_text(color=&quot;black&quot;),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())
diff_gene_transcript_cortex_density_plot</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-2-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-2-1">
Past versions of unnamed-chunk-2-1.png
</button>
</p>
<div id="fig-unnamed-chunk-2-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-2-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="figure-1d" class="section level4">
<h4>Figure 1D</h4>
<p>We performed the cell type enrichment analysis using EWCE package in
R using different set of gene lists such as upregulated genes in
hippocampus etc. and the heatmap was generated using “superheat” package
in R.</p>
<p>We are not showing the plotting scripts for this figure.</p>
</div>
<div id="figure-1e" class="section level4">
<h4>Figure 1E</h4>
<p>We will read the data for GO terms which has been already cleaned up
by taking the terms for both gene and transcript expression profiles and
preparing the table with -log10FDR values from DAVID analysis.</p>
<p>We will show a heatmap for GO terms for upregulated genes and
transcripts from hippocampus only. The others are done similarly.</p>
<pre class="r"><code>human_hippo_gene_transcript_DAVID_up&lt;-read.xlsx(&quot;data/GO_terms_figure1E.xlsx&quot;,sheet=1)
row.names(human_hippo_gene_transcript_DAVID_up)&lt;-human_hippo_gene_transcript_DAVID_up$Term
human_hippo_gene_transcript_DAVID_up_filt&lt;-as.data.frame(human_hippo_gene_transcript_DAVID_up[,-c(1)])
f3 = colorRamp2(seq(min(human_hippo_gene_transcript_DAVID_up_filt), max(human_hippo_gene_transcript_DAVID_up_filt), length = 3), c(&quot;#EEEEEE&quot;,&quot;blue&quot;, &quot;red&quot;), space = &quot;RGB&quot;)

Heatmap(human_hippo_gene_transcript_DAVID_up_filt, col = f3, cluster_rows = FALSE,cluster_columns = FALSE,
        row_order = order(as.numeric(gsub(&quot;row&quot;, &quot;&quot;, rownames(human_hippo_gene_transcript_DAVID_up_filt)))), 
        column_order = order(as.numeric(gsub(&quot;column&quot;, &quot;&quot;, colnames(human_hippo_gene_transcript_DAVID_up_filt)))),
        row_names_gp = gpar(fontsize = 8),column_names_gp = gpar(fontsize=9),
        column_names_centered = TRUE,heatmap_width = unit(3, &quot;in&quot;))</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-3-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-3-1">
Past versions of unnamed-chunk-3-1.png
</button>
</p>
<div id="fig-unnamed-chunk-3-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-3-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="supplementary-figure-1a" class="section level4">
<h4>Supplementary Figure 1A</h4>
<p>The data for preparing this figure in the folder “data” on the github
repository.</p>
</div>
<div id="supplementary-figure-1b" class="section level4">
<h4>Supplementary figure 1B</h4>
<p>This figure is shown in the differential expression analysis for the
hippocampus in Data analysis and results section.</p>
</div>
<div id="supplementary-figure-1c" class="section level4">
<h4>Supplementary Figure 1C</h4>
<div
id="first-we-need-to-add-a-column-to-the-result-obtained-from-the-differential-expression-analysis-for-gene-expression-to-provide-a-value-for-chr21-originated-genes-upregulated-downregulated-and-non-differentially-expressed."
class="section level5">
<h5>First, we need to add a column to the result obtained from the
differential expression analysis for gene expression, to provide a value
for chr21 originated genes, upregulated, downregulated and non
differentially expressed.</h5>
</div>
<div
id="after-preparing-the-volcano-plots-for-individual-region-they-were-overlapped-in-adobe-illustrator-to-prepare-the-final-figure."
class="section level5">
<h5>After preparing the volcano plots for individual region, they were
overlapped in Adobe Illustrator to prepare the final figure.</h5>
</div>
<div id="hippocampus-non-chr21-genes" class="section level5">
<h5>Hippocampus non chr21 genes</h5>
<pre class="r"><code>non_chr21_hippo_gene&lt;-read.table(file=&quot;data/non_chr21_hippo_gene.txt&quot;, header=T, row.names=1)
volcano_non_chr21_hippo_gene&lt;- ggplot(non_chr21_hippo_gene, aes(x = logFC, y = -log10(FDR))) +
  geom_point(aes(color =  factor(Significance)), alpha=1, size=2.0) +
  xlim(-3,4)+
  ylim(0, 13)+
  scale_color_manual(values = c(&quot;#219EBC&quot;,&quot;#CED4DA&quot;,&quot;#DB5256&quot;)) +
  xlab(&quot;log2 fold change&quot;) + ylab(&quot;-log10(FDR)&quot;)+
  theme_bw() +
  theme(text=element_text(size=18),axis.text.x = element_text(color = &quot;black&quot;),axis.text.y = element_text(color=&quot;black&quot;),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())
volcano_non_chr21_hippo_gene+geom_hline(yintercept=1.30, linetype=&quot;dashed&quot;, color = &quot;black&quot;)+
  geom_vline(xintercept = 0,linetype=&quot;dashed&quot;,color=&quot;black&quot;)</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-4-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-4-1">
Past versions of unnamed-chunk-4-1.png
</button>
</p>
<div id="fig-unnamed-chunk-4-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-4-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="hippocampus-chr21-genes" class="section level5">
<h5>Hippocampus chr21 genes</h5>
<pre class="r"><code>chr21_hippo_gene&lt;-read.table(file=&quot;data/chr21_hippo_gene.txt&quot;, header=T, row.names=1)
volcano_chr21_hippo_gene&lt;- ggplot(chr21_hippo_gene, aes(x = logFC, y = -log10(FDR))) +
  geom_point(aes(color =  factor(Significance)), alpha=1, size=2.0) +
  xlim(-3,4)+
  ylim(0, 13)+
  scale_color_manual(values = c(&quot;BLACK&quot;)) +
  xlab(&quot;log2 fold change&quot;) + ylab(&quot;-log10(FDR)&quot;)+
  theme_bw() +
  theme(text=element_text(size=18),axis.text.x = element_text(color = &quot;black&quot;),axis.text.y = element_text(color=&quot;black&quot;),
        panel.grid.major = element_blank(), panel.grid.minor = element_blank(),
        panel.background = element_blank())
volcano_chr21_hippo_gene+geom_hline(yintercept=1.30, linetype=&quot;dashed&quot;, color = &quot;black&quot;)+
  geom_vline(xintercept = 0,linetype=&quot;dashed&quot;,color=&quot;black&quot;)</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-5-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-5-1">
Past versions of unnamed-chunk-5-1.png
</button>
</p>
<div id="fig-unnamed-chunk-5-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-5-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="supplementary-figure-1e" class="section level4">
<h4>Supplementary Figure 1E</h4>
<div id="genome-wide-dysregulation-domains" class="section level5">
<h5>Genome wide dysregulation domains</h5>
<pre class="r"><code>results_hippo_RNA&lt;- readRDS(&quot;output/results_hippo_RNA.rds&quot;)
sigGenes &lt;- results_hippo_RNA
tx&lt;-TxDb.Hsapiens.UCSC.hg38.knownGene
exo &lt;- exonsBy(tx,&quot;gene&quot;)
exoRanges &lt;- unlist(range(exo))
sigRegions &lt;- exoRanges[na.omit(match(sigGenes$ENTREZID, names(exoRanges)))]
mcols(sigRegions) &lt;- sigGenes[match(names(sigRegions), sigGenes$ENTREZID),]
sigRegions</code></pre>
<pre><code>GRanges object with 14146 ranges and 12 metadata columns:
                    seqnames            ranges strand |     logFC    logCPM
                       &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt;
      26155             chr1     944203-959309      - |  0.286287  5.123650
     339451             chr1     960584-965719      + |  1.444466  1.134020
      57801             chr1    998962-1000172      - |  0.540323  0.786063
       9636             chr1   1001138-1014540      + |  0.339285  1.562178
     375790             chr1   1020120-1056118      + |  1.428524  4.581598
        ...              ...               ...    ... .       ...       ...
      23181            chr21 46458891-46570015      + |  0.921378   6.25343
       6285            chr21 46598604-46605208      - |  0.299546   7.87716
       3275            chr21 46635595-46665124      + |  0.485232   6.94981
     389831 chrUn_GL000195v1       42939-49164      - | -1.191513   2.15811
  102724788            chr22 18912777-18936553      - |  0.734929   2.61639
                   LR      PValue         FDR         ENSEMBL    ENTREZID
            &lt;numeric&gt;   &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt;
      26155  1.780455 1.82093e-01 0.294401021 ENSG00000188976       26155
     339451 12.014082 5.28001e-04 0.003696271 ENSG00000187961      339451
      57801  1.091323 2.96178e-01 0.423355905 ENSG00000188290       57801
       9636  0.522737 4.69676e-01 0.589624513 ENSG00000187608        9636
     375790 16.370628 5.20862e-05 0.000684321 ENSG00000188157      375790
        ...       ...         ...         ...             ...         ...
      23181  33.96882 5.60024e-09 7.80794e-07 ENSG00000160305       23181
       6285   3.24890 7.14715e-02 1.45929e-01 ENSG00000160307        6285
       3275  11.57174 6.69616e-04 4.42441e-03 ENSG00000160310        3275
     389831   6.09842 1.35303e-02 4.21130e-02 ENSG00000276256      389831
  102724788   1.28594 2.56797e-01 3.80327e-01 ENSG00000277196   102724788
                  SYMBOL               GENENAME         CHR     UNIPROT
             &lt;character&gt;            &lt;character&gt; &lt;character&gt; &lt;character&gt;
      26155        NOC2L NOC2 like nucleolar ..           1      Q9Y3T9
     339451       KLHL17 kelch like family me..           1      Q6TDP4
      57801         HES4 hes family bHLH tran..           1      E9PB28
       9636        ISG15 ISG15 ubiquitin like..           1      P05161
     375790         AGRN                  agrin           1      O00468
        ...          ...                    ...         ...         ...
      23181        DIP2A disco interacting pr..          21      Q14689
       6285        S100B S100 calcium binding..          21  A0A0S2Z4C5
       3275        PRMT2 protein arginine met..          21      P55345
     389831    LOC389831 uncharacterized LOC3..          14        &lt;NA&gt;
  102724788 LOC102724788 proline dehydrogenas..          22        &lt;NA&gt;
                   ALIAS
             &lt;character&gt;
      26155        NET15
     339451           AF
      57801      bHLHb42
       9636         G1P2
     375790        AGRIN
        ...          ...
      23181    C21orf106
       6285          NEF
       3275      HRMT1L1
     389831    LOC389831
  102724788 LOC102724788
  -------
  seqinfo: 595 sequences (1 circular) from hg38 genome</code></pre>
<pre class="r"><code>sigRegions[order(sigRegions$LR,decreasing = TRUE)]</code></pre>
<pre><code>GRanges object with 14146 ranges and 12 metadata columns:
         seqnames              ranges strand |        logFC    logCPM
            &lt;Rle&gt;           &lt;IRanges&gt;  &lt;Rle&gt; |    &lt;numeric&gt; &lt;numeric&gt;
   64005     chr7   44962662-44979088      - |      3.67879  0.606683
    7052    chr20   38127385-38166578      - |      2.25840  4.274056
    3339     chr1   21822244-21937310      - |      2.44714  3.740847
      12    chr14   94592058-94624646      + |      3.16526  0.988412
    3691    chr17   75721328-75757818      + |      2.18007  5.528861
     ...      ...                 ...    ... .          ...       ...
   27079    chr15   40569299-40574949      + | -0.000338077  2.074500
   26290    chr12     4669741-4851003      + |  0.000251939  3.436495
  196441    chr12   71609599-71667725      - | -0.000171008  6.947769
   54623    chr19   39385629-39391154      - |  0.000149507  5.986378
  285643     chr5 155013755-155018141      + |  0.000296206  0.172698
                  LR      PValue         FDR         ENSEMBL    ENTREZID
           &lt;numeric&gt;   &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt;
   64005     66.0467 4.40354e-16 2.67493e-12 ENSG00000136286       64005
    7052     65.9522 4.61983e-16 2.67493e-12 ENSG00000198959        7052
    3339     65.5580 5.64291e-16 2.67493e-12 ENSG00000142798        3339
      12     64.9642 7.62729e-16 2.71169e-12 ENSG00000196136          12
    3691     64.2538 1.09383e-15 3.11108e-12 ENSG00000132470        3691
     ...         ...         ...         ...             ...         ...
   27079 1.33593e-06    0.999078    0.999353 ENSG00000166133       27079
   26290 1.15559e-06    0.999142    0.999353 ENSG00000130035       26290
  196441 9.56964e-07    0.999219    0.999360 ENSG00000133858      196441
   54623 7.65266e-07    0.999302    0.999372 ENSG00000006712       54623
  285643 4.61344e-07    0.999458    0.999458 ENSG00000226650      285643
              SYMBOL               GENENAME         CHR     UNIPROT       ALIAS
         &lt;character&gt;            &lt;character&gt; &lt;character&gt; &lt;character&gt; &lt;character&gt;
   64005       MYO1G              myosin IG           7      B0I1T2         HA2
    7052        TGM2     transglutaminase 2          20      P21980        G(h)
    3339       HSPG2 heparan sulfate prot..           1      P98160        HSPG
      12    SERPINA3 serpin family A memb..          14  A0A024R6P0        AACT
    3691       ITGB4 integrin subunit bet..          17  A0A024R8T0       CD104
     ...         ...                    ...         ...         ...         ...
   27079      RPUSD2 RNA pseudouridine sy..          15      Q8IZ73    C15orf19
   26290      GALNT8 polypeptide N-acetyl..          12      Q9NY28   GALNAC-T8
  196441      ZFC3H1 zinc finger C3H1-typ..          12      O60293     CCDC131
   54623        PAF1 PAF1 homolog, Paf1/R..          19      Q8N7H5    F23149_1
  285643       KIF4B kinesin family membe..           5      B4DYE2       KIF4B
  -------
  seqinfo: 595 sequences (1 circular) from hg38 genome</code></pre>
<pre class="r"><code>filtered.sigRegions &lt;- sigRegions[!is.na(sigRegions$FDR)]
log.pval &lt;- -log10(filtered.sigRegions$FDR)
mcols(filtered.sigRegions)$log.pval &lt;- log.pval
filtered.sigRegions</code></pre>
<pre><code>GRanges object with 14146 ranges and 13 metadata columns:
                    seqnames            ranges strand |     logFC    logCPM
                       &lt;Rle&gt;         &lt;IRanges&gt;  &lt;Rle&gt; | &lt;numeric&gt; &lt;numeric&gt;
      26155             chr1     944203-959309      - |  0.286287  5.123650
     339451             chr1     960584-965719      + |  1.444466  1.134020
      57801             chr1    998962-1000172      - |  0.540323  0.786063
       9636             chr1   1001138-1014540      + |  0.339285  1.562178
     375790             chr1   1020120-1056118      + |  1.428524  4.581598
        ...              ...               ...    ... .       ...       ...
      23181            chr21 46458891-46570015      + |  0.921378   6.25343
       6285            chr21 46598604-46605208      - |  0.299546   7.87716
       3275            chr21 46635595-46665124      + |  0.485232   6.94981
     389831 chrUn_GL000195v1       42939-49164      - | -1.191513   2.15811
  102724788            chr22 18912777-18936553      - |  0.734929   2.61639
                   LR      PValue         FDR         ENSEMBL    ENTREZID
            &lt;numeric&gt;   &lt;numeric&gt;   &lt;numeric&gt;     &lt;character&gt; &lt;character&gt;
      26155  1.780455 1.82093e-01 0.294401021 ENSG00000188976       26155
     339451 12.014082 5.28001e-04 0.003696271 ENSG00000187961      339451
      57801  1.091323 2.96178e-01 0.423355905 ENSG00000188290       57801
       9636  0.522737 4.69676e-01 0.589624513 ENSG00000187608        9636
     375790 16.370628 5.20862e-05 0.000684321 ENSG00000188157      375790
        ...       ...         ...         ...             ...         ...
      23181  33.96882 5.60024e-09 7.80794e-07 ENSG00000160305       23181
       6285   3.24890 7.14715e-02 1.45929e-01 ENSG00000160307        6285
       3275  11.57174 6.69616e-04 4.42441e-03 ENSG00000160310        3275
     389831   6.09842 1.35303e-02 4.21130e-02 ENSG00000276256      389831
  102724788   1.28594 2.56797e-01 3.80327e-01 ENSG00000277196   102724788
                  SYMBOL               GENENAME         CHR     UNIPROT
             &lt;character&gt;            &lt;character&gt; &lt;character&gt; &lt;character&gt;
      26155        NOC2L NOC2 like nucleolar ..           1      Q9Y3T9
     339451       KLHL17 kelch like family me..           1      Q6TDP4
      57801         HES4 hes family bHLH tran..           1      E9PB28
       9636        ISG15 ISG15 ubiquitin like..           1      P05161
     375790         AGRN                  agrin           1      O00468
        ...          ...                    ...         ...         ...
      23181        DIP2A disco interacting pr..          21      Q14689
       6285        S100B S100 calcium binding..          21  A0A0S2Z4C5
       3275        PRMT2 protein arginine met..          21      P55345
     389831    LOC389831 uncharacterized LOC3..          14        &lt;NA&gt;
  102724788 LOC102724788 proline dehydrogenas..          22        &lt;NA&gt;
                   ALIAS  log.pval
             &lt;character&gt; &lt;numeric&gt;
      26155        NET15  0.531061
     339451           AF  2.432236
      57801      bHLHb42  0.373294
       9636         G1P2  0.229424
     375790        AGRIN  3.164740
        ...          ...       ...
      23181    C21orf106  6.107464
       6285          NEF  0.835858
       3275      HRMT1L1  2.354144
     389831    LOC389831  1.375584
  102724788 LOC102724788  0.419842
  -------
  seqinfo: 595 sequences (1 circular) from hg38 genome</code></pre>
<pre class="r"><code>range(filtered.sigRegions$logFC)</code></pre>
<pre><code>[1] -2.372755  3.678790</code></pre>
<pre class="r"><code>fc.ymax &lt;- ceiling(max(abs(range(filtered.sigRegions$logFC))))
fc.ymin &lt;- -fc.ymax
col.over &lt;- &quot;#DB5256&quot;
col.under &lt;- &quot;#219EBC&quot;
col.non &lt;- &quot;#95A3A6&quot;
#sign.col &lt;- rep(col.over, length(filtered.sigRegions$FDR&lt;0.05))
#sign.col[filtered.sigRegions$FDR&gt;=0.05] &lt;- col.under
sign.col &lt;- rep(col.over, length(filtered.sigRegions$logFC&gt;0.3 &amp; filtered.sigRegions$FDR&lt;0.05))
sign.col[filtered.sigRegions$FDR&lt;0.05 &amp; filtered.sigRegions$logFC&lt;(-0.3)] &lt;- col.under
sign.col[filtered.sigRegions$FDR&gt;=0.05] &lt;- col.non
pp &lt;- getDefaultPlotParams(plot.type = 2)</code></pre>
</div>
<div id="chromosome-1" class="section level5">
<h5>Chromosome 1</h5>
<pre class="r"><code>filtered.sigRegions_chr1&lt;-subset(filtered.sigRegions,filtered.sigRegions$CHR==&quot;1&quot;)
fit2 &lt;- loess(filtered.sigRegions_chr1$logFC ~ start(filtered.sigRegions_chr1),span = 0.03)
tibbe_chr1&lt;-tibble(logFC=filtered.sigRegions_chr1$logFC,FDR=filtered.sigRegions_chr1$FDR,ENSEMBL=filtered.sigRegions_chr1$ENSEMBL,SYMBOL=filtered.sigRegions_chr1$SYMBOL,start=start(filtered.sigRegions_chr1))
bootstrap_samples &lt;- bootstraps(tibbe_chr1, times = 25)
summary(fit2)</code></pre>
<pre><code>Call:
loess(formula = filtered.sigRegions_chr1$logFC ~ start(filtered.sigRegions_chr1), 
    span = 0.03)

Number of Observations: 1421 
Equivalent Number of Parameters: 99.89 
Residual Standard Error: 0.4993 
Trace of smoother matrix: 109.96  (exact)

Control settings:
  span     :  0.03 
  degree   :  2 
  family   :  gaussian
  surface  :  interpolate     cell = 0.2
  normalize:  TRUE
 parametric:  FALSE
drop.square:  FALSE </code></pre>
<pre class="r"><code>lowess_fit &lt;- function(split){
lowess(analysis(split)$start,analysis(split)$logFC, f=3/100)
}
bootstrap_samples %&gt;%
mutate(lowess_points = map(splits, lowess_fit)) %&gt;%
select(id, lowess_points) %&gt;%
mutate_if(is.list, map, as_tibble) %&gt;%
unnest(cols = c(lowess_points))</code></pre>
<pre><code># A tibble: 35,525 × 3
   id                x     y
   &lt;chr&gt;         &lt;dbl&gt; &lt;dbl&gt;
 1 Bootstrap01  944203 0.789
 2 Bootstrap01  960584 0.787
 3 Bootstrap01  998962 0.785
 4 Bootstrap01 1001138 0.785
 5 Bootstrap01 1081818 0.779
 6 Bootstrap01 1081818 0.779
 7 Bootstrap01 1216909 0.769
 8 Bootstrap01 1216909 0.769
 9 Bootstrap01 1216909 0.769
10 Bootstrap01 1232237 0.768
# … with 35,515 more rows</code></pre>
<pre class="r"><code>lowess_25 &lt;- bootstrap_samples %&gt;%
mutate(lowess_points = map(splits, lowess_fit)) %&gt;%
select(id, lowess_points) %&gt;%
mutate_if(is.list, map, as_tibble) %&gt;%
unnest(cols = c(lowess_points)) %&gt;%
distinct()</code></pre>
<pre class="r"><code>kp1 &lt;- plotKaryotype(genome = &quot;hg38&quot;,chromosomes = &quot;chr1&quot;,plot.type = 2, plot.params = pp) %&gt;%
kpAddMainTitle(main = &quot;Chr1&quot;) %&gt;%
kpPoints(data=filtered.sigRegions_chr1, y=filtered.sigRegions_chr1$logFC, ymax=fc.ymax, ymin=fc.ymin, col=sign.col) %&gt;%
kpPlotLoess(data=filtered.sigRegions_chr1, y=filtered.sigRegions_chr1$logFC, ymin=fc.ymin, ymax=fc.ymax,span = 0.03, conf.interval = (0.95),col=&quot;#000000&quot;) %&gt;%
kpAxis(ymax=fc.ymax, ymin=fc.ymin) %&gt;%
kpAddLabels(labels = &quot;logFC&quot;, srt=90, pos=1, label.margin = 0.06, ymax=fc.ymax, ymin=fc.ymin) %&gt;%
kpPlotDensity(data=exoRanges, window.size = 10e4, data.panel = 2)</code></pre>
<pre><code>Warning in text.default(x = x, y = y, labels = labels, pos = pos, offset = 0, :
&quot;ymax&quot; is not a graphical parameter</code></pre>
<pre><code>Warning in text.default(x = x, y = y, labels = labels, pos = pos, offset = 0, :
&quot;ymin&quot; is not a graphical parameter</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-8-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-8-1">
Past versions of unnamed-chunk-8-1.png
</button>
</p>
<div id="fig-unnamed-chunk-8-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-8-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
<div id="chromosome-11" class="section level5">
<h5>Chromosome 11</h5>
<pre class="r"><code>filtered.sigRegions_chr11&lt;-subset(filtered.sigRegions,filtered.sigRegions$CHR==&quot;11&quot;)
fit2 &lt;- loess(filtered.sigRegions_chr11$logFC ~ start(filtered.sigRegions_chr11),span = 0.03)
tibbe_chr11&lt;-tibble(logFC=filtered.sigRegions_chr11$logFC,FDR=filtered.sigRegions_chr11$FDR,ENSEMBL=filtered.sigRegions_chr11$ENSEMBL,SYMBOL=filtered.sigRegions_chr11$SYMBOL,start=start(filtered.sigRegions_chr11))
bootstrap_samples &lt;- bootstraps(tibbe_chr11, times = 25)
summary(fit2)</code></pre>
<pre><code>Call:
loess(formula = filtered.sigRegions_chr11$logFC ~ start(filtered.sigRegions_chr11), 
    span = 0.03)

Number of Observations: 790 
Equivalent Number of Parameters: 105.42 
Residual Standard Error: 0.4993 
Trace of smoother matrix: 115.7  (exact)

Control settings:
  span     :  0.03 
  degree   :  2 
  family   :  gaussian
  surface  :  interpolate     cell = 0.2
  normalize:  TRUE
 parametric:  FALSE
drop.square:  FALSE </code></pre>
<pre class="r"><code>lowess_fit &lt;- function(split){
lowess(analysis(split)$start,analysis(split)$logFC, f=3/100)
}
bootstrap_samples %&gt;%
mutate(lowess_points = map(splits, lowess_fit)) %&gt;%
select(id, lowess_points) %&gt;%
mutate_if(is.list, map, as_tibble) %&gt;%
unnest(cols = c(lowess_points))</code></pre>
<pre><code># A tibble: 19,750 × 3
   id               x     y
   &lt;chr&gt;        &lt;dbl&gt; &lt;dbl&gt;
 1 Bootstrap01 236966 1.00 
 2 Bootstrap01 289126 0.994
 3 Bootstrap01 307631 0.990
 4 Bootstrap01 308408 0.990
 5 Bootstrap01 369499 0.978
 6 Bootstrap01 369499 0.978
 7 Bootstrap01 369499 0.978
 8 Bootstrap01 448268 0.962
 9 Bootstrap01 448268 0.962
10 Bootstrap01 494512 0.953
# … with 19,740 more rows</code></pre>
<pre class="r"><code>lowess_25 &lt;- bootstrap_samples %&gt;%
mutate(lowess_points = map(splits, lowess_fit)) %&gt;%
select(id, lowess_points) %&gt;%
mutate_if(is.list, map, as_tibble) %&gt;%
unnest(cols = c(lowess_points)) %&gt;%
distinct()</code></pre>
<pre class="r"><code>kp11 &lt;- plotKaryotype(genome = &quot;hg38&quot;,chromosomes = &quot;chr11&quot;,plot.type = 2, plot.params = pp) %&gt;%
kpAddMainTitle(main = &quot;Chr11&quot;) %&gt;%
kpPoints(data=filtered.sigRegions_chr11, y=filtered.sigRegions_chr11$logFC, ymax=fc.ymax, ymin=fc.ymin, col=sign.col) %&gt;%
kpPlotLoess(data=filtered.sigRegions_chr11, y=filtered.sigRegions_chr11$logFC, ymin=fc.ymin, ymax=fc.ymax,span = 0.03, conf.interval = (0.95),col=&quot;#000000&quot;) %&gt;%
kpAxis(ymax=fc.ymax, ymin=fc.ymin) %&gt;%
kpAddLabels(labels = &quot;logFC&quot;, srt=90, pos=1, label.margin = 0.06, ymax=fc.ymax, ymin=fc.ymin) %&gt;%
kpPlotDensity(data=exoRanges, window.size = 10e4, data.panel = 2)</code></pre>
<pre><code>Warning in text.default(x = x, y = y, labels = labels, pos = pos, offset = 0, :
&quot;ymax&quot; is not a graphical parameter</code></pre>
<pre><code>Warning in text.default(x = x, y = y, labels = labels, pos = pos, offset = 0, :
&quot;ymin&quot; is not a graphical parameter</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-10-1.png" width="768" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-10-1">
Past versions of unnamed-chunk-10-1.png
</button>
</p>
<div id="fig-unnamed-chunk-10-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-10-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
</div>
</div>
<div id="supplementary-figure-1f" class="section level4">
<h4>Supplementary Figure 1F</h4>
<pre class="r"><code>diff_gene_transcript_cortex&lt;- read.table(file= &quot;data/diff_gene_transcript_cortex.txt&quot;, header=T, row.names=1)
GABA_family_hippo&lt;-read.xlsx(&quot;data/GABA_AMPA_family.xlsx&quot;,sheet=1)
GABA_01 &lt;- ggplot(GABA_family_hippo, aes(CPM, Symbol,fill=Group))+
  geom_bar(stat=&quot;identity&quot;,color=&quot;black&quot;, width = 0.7, position = &quot;dodge&quot;) +
  geom_errorbar(aes(xmin=CPM-sem, xmax=CPM+sem), width=.1,position=position_dodge(.7))+
  scale_fill_manual(values = c(&quot;#B9D47A&quot;,&quot;#C2A3DC&quot;)) +
  xlim(0,700)+
  theme_bw()+
  theme(legend.position=&quot;none&quot;,axis.text.x = element_text(angle=90, vjust=0.6,color =&quot;black&quot;),text=element_text(size=10),
        axis.text.y = element_text(color=&quot;black&quot;),panel.background = element_blank(),panel.grid.major = element_blank(),
        panel.grid.minor = element_blank()) +
  labs(x=&quot;Counts per Million&quot;,y=&quot;GABA subunits&quot;)
GABA_01</code></pre>
<p><img src="figure/Figure1.Rmd/unnamed-chunk-11-1.png" width="672" style="display: block; margin: auto;" /></p>
<p>
<button type="button" class="btn btn-default btn-xs btn-workflowr btn-workflowr-fig" data-toggle="collapse" data-target="#fig-unnamed-chunk-11-1">
Past versions of unnamed-chunk-11-1.png
</button>
</p>
<div id="fig-unnamed-chunk-11-1" class="collapse">
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<a href="https://github.com/neurodevdisorder/HumanMultiomics/blob/dd963a3aeb00790c0320cfc4dc6aad60cc3f4fc0/docs/figure/Figure1.Rmd/unnamed-chunk-11-1.png" target="_blank">dd963a3</a>
</td>
<td>
mohit-rastogi
</td>
<td>
2022-09-26
</td>
</tr>
</tbody>
</table>
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span>
Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.0 (2021-05-18)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Catalina 10.15.7

Matrix products: default
BLAS:   /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRblas.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/4.1/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] grid      stats4    stats     graphics  grDevices utils     datasets 
[8] methods   base     

other attached packages:
 [1] superheat_0.1.0                         
 [2] magrittr_2.0.3                          
 [3] rsample_1.1.0                           
 [4] circlize_0.4.15                         
 [5] ComplexHeatmap_2.8.0                    
 [6] pathview_1.32.0                         
 [7] forcats_0.5.2                           
 [8] stringr_1.4.1                           
 [9] purrr_0.3.4                             
[10] readr_2.1.2                             
[11] tidyr_1.2.1                             
[12] tibble_3.1.8                            
[13] tidyverse_1.3.2                         
[14] karyoploteR_1.18.0                      
[15] regioneR_1.24.0                         
[16] ggridges_0.5.3                          
[17] enrichplot_1.12.3                       
[18] clusterProfiler_4.0.5                   
[19] ggbio_1.40.0                            
[20] rtracklayer_1.52.1                      
[21] TxDb.Hsapiens.UCSC.hg38.knownGene_3.13.0
[22] GenomicFeatures_1.44.2                  
[23] GenomicRanges_1.44.0                    
[24] GenomeInfoDb_1.28.4                     
[25] pcaExplorer_2.18.0                      
[26] org.Hs.eg.db_3.13.0                     
[27] AnnotationDbi_1.54.1                    
[28] IRanges_2.28.0                          
[29] S4Vectors_0.32.3                        
[30] Biobase_2.54.0                          
[31] BiocGenerics_0.40.0                     
[32] DBI_1.1.3                               
[33] ggrepel_0.9.1                           
[34] openxlsx_4.2.5                          
[35] edgeR_3.34.1                            
[36] limma_3.50.1                            
[37] dplyr_1.0.10                            
[38] ggplot2_3.3.6                           
[39] workflowr_1.7.0                         

loaded via a namespace (and not attached):
  [1] Hmisc_4.7-1                 ps_1.7.1                   
  [3] Rsamtools_2.8.0             foreach_1.5.2              
  [5] rprojroot_2.0.3             crayon_1.5.1               
  [7] MASS_7.3-58.1               nlme_3.1-159               
  [9] backports_1.4.1             reprex_2.0.2               
 [11] GOSemSim_2.18.1             rlang_1.0.5                
 [13] XVector_0.32.0              readxl_1.4.1               
 [15] SparseM_1.81                callr_3.7.2                
 [17] filelock_1.0.2              GOstats_2.58.0             
 [19] BiocParallel_1.28.3         rjson_0.2.21               
 [21] bit64_4.0.5                 glue_1.6.2                 
 [23] pheatmap_1.0.12             rngtools_1.5.2             
 [25] parallel_4.1.0              processx_3.7.0             
 [27] shinyAce_0.4.2              shinydashboard_0.7.2       
 [29] DOSE_3.18.3                 haven_2.5.1                
 [31] tidyselect_1.1.2            SummarizedExperiment_1.22.0
 [33] XML_3.99-0.10               GenomicAlignments_1.28.0   
 [35] xtable_1.8-4                evaluate_0.16              
 [37] cli_3.4.0                   zlibbioc_1.40.0            
 [39] rstudioapi_0.14             furrr_0.3.1                
 [41] whisker_0.4                 bslib_0.4.0                
 [43] rpart_4.1.16                fastmatch_1.1-3            
 [45] ensembldb_2.16.4            treeio_1.16.2              
 [47] shiny_1.7.2                 xfun_0.33                  
 [49] clue_0.3-61                 cluster_2.1.4              
 [51] tidygraph_1.2.2             TSP_1.2-1                  
 [53] KEGGREST_1.32.0             biovizBase_1.40.0          
 [55] threejs_0.3.3               ape_5.6-2                  
 [57] listenv_0.8.0               dendextend_1.16.0          
 [59] future_1.28.0               Biostrings_2.60.2          
 [61] png_0.1-7                   reshape_0.8.9              
 [63] withr_2.5.0                 shinyBS_0.61.1             
 [65] bitops_1.0-7                ggforce_0.3.4              
 [67] RBGL_1.68.0                 plyr_1.8.7                 
 [69] cellranger_1.1.0            GSEABase_1.54.0            
 [71] AnnotationFilter_1.16.0     pillar_1.8.1               
 [73] GlobalOptions_0.1.2         cachem_1.0.6               
 [75] fs_1.5.2                    GetoptLong_1.0.5           
 [77] vctrs_0.4.1                 ellipsis_0.3.2             
 [79] generics_0.1.3              NMF_0.24.0                 
 [81] tools_4.1.0                 foreign_0.8-82             
 [83] munsell_0.5.0               tweenr_2.0.2               
 [85] fgsea_1.18.0                DelayedArray_0.18.0        
 [87] fastmap_1.1.0               compiler_4.1.0             
 [89] httpuv_1.6.6                pkgmaker_0.32.2            
 [91] plotly_4.10.0               GenomeInfoDbData_1.2.6     
 [93] gridExtra_2.3               lattice_0.20-45            
 [95] deldir_1.0-6                AnnotationForge_1.34.1     
 [97] utf8_1.2.2                  later_1.3.0                
 [99] BiocFileCache_2.0.0         jsonlite_1.8.0             
[101] GGally_2.1.2                scales_1.2.1               
[103] graph_1.70.0                tidytree_0.4.0             
[105] genefilter_1.74.1           lazyeval_0.2.2             
[107] promises_1.2.0.1            doParallel_1.0.17          
[109] latticeExtra_0.6-30         checkmate_2.1.0            
[111] rmarkdown_2.16              cowplot_1.1.1              
[113] webshot_0.5.3               dichromat_2.0-0.1          
[115] downloader_0.4              BSgenome_1.60.0            
[117] igraph_1.3.4                survival_3.4-0             
[119] yaml_2.3.5                  htmltools_0.5.3            
[121] memoise_2.0.1               VariantAnnotation_1.38.0   
[123] BiocIO_1.2.0                locfit_1.5-9.6             
[125] seriation_1.3.6             graphlayouts_0.8.1         
[127] viridisLite_0.4.1           digest_0.6.29              
[129] assertthat_0.2.1            mime_0.12                  
[131] rappdirs_0.3.3              registry_0.5-1             
[133] RSQLite_2.2.17              yulab.utils_0.0.5          
[135] data.table_1.14.2           blob_1.2.3                 
[137] labeling_0.4.2              splines_4.1.0              
[139] Formula_1.2-4               Cairo_1.6-0                
[141] googledrive_2.0.0           OrganismDbi_1.34.0         
[143] ProtGenerics_1.26.0         RCurl_1.98-1.8             
[145] broom_1.0.1                 hms_1.1.2                  
[147] modelr_0.1.9                colorspace_2.0-3           
[149] base64enc_0.1-3             BiocManager_1.30.18        
[151] shape_1.4.6                 aplot_0.1.7                
[153] nnet_7.3-17                 sass_0.4.2                 
[155] Rcpp_1.0.9                  fansi_1.0.3                
[157] tzdb_0.3.0                  parallelly_1.32.1          
[159] R6_2.5.1                    lifecycle_1.0.2            
[161] zip_2.2.1                   curl_4.3.2                 
[163] googlesheets4_1.0.1         jquerylib_0.1.4            
[165] DO.db_2.9                   Matrix_1.4-1               
[167] qvalue_2.24.0               RColorBrewer_1.1-3         
[169] iterators_1.0.14            topGO_2.44.0               
[171] htmlwidgets_1.5.4           bamsignals_1.24.0          
[173] polyclip_1.10-0             biomaRt_2.48.3             
[175] crosstalk_1.2.0             shadowtext_0.1.2           
[177] gridGraphics_0.5-1          rvest_1.0.3                
[179] globals_0.16.1              htmlTable_2.4.1            
[181] patchwork_1.1.2             KEGGgraph_1.52.0           
[183] codetools_0.2-18            matrixStats_0.62.0         
[185] lubridate_1.8.0             GO.db_3.13.0               
[187] getPass_0.2-2               prettyunits_1.1.1          
[189] dbplyr_2.2.1                gridBase_0.4-7             
[191] gtable_0.3.1                git2r_0.30.1               
[193] highr_0.9                   ggfun_0.0.7                
[195] httr_1.4.4                  stringi_1.7.8              
[197] progress_1.2.2              reshape2_1.4.4             
[199] farver_2.1.1                heatmaply_1.3.0            
[201] annotate_1.70.0             viridis_0.6.2              
[203] Rgraphviz_2.36.0            ggtree_3.0.4               
[205] DT_0.25.1                   xml2_1.3.3                 
[207] bezier_1.1.2                restfulr_0.0.15            
[209] interp_1.1-3                geneplotter_1.70.0         
[211] ggplotify_0.1.0             Category_2.58.0            
[213] DESeq2_1.32.0               bit_4.0.4                  
[215] scatterpie_0.1.8            jpeg_0.1-9                 
[217] MatrixGenerics_1.4.3        ggraph_2.0.6               
[219] pkgconfig_2.0.3             gargle_1.2.1               
[221] knitr_1.40                 </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
https://docs.mathjax.org/en/latest/web/configuration.html. This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3,h4",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
